[2024-05-12T10:58:49.190+0000] {processor.py:161} INFO - Started process (PID=87) to work on /opt/airflow/dags/script.py
[2024-05-12T10:58:49.192+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/script.py for tasks to queue
[2024-05-12T10:58:49.203+0000] {logging_mixin.py:188} INFO - [2024-05-12T10:58:49.200+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/script.py
[2024-05-12T10:58:49.244+0000] {logging_mixin.py:188} INFO - [2024-05-12T10:58:49.239+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/script.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/script.py", line 19
    def transform_data():
    ^^^
IndentationError: expected an indented block after function definition on line 16
[2024-05-12T10:58:49.283+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/script.py
[2024-05-12T10:58:49.355+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/script.py took 0.184 seconds
[2024-05-12T10:59:20.634+0000] {processor.py:161} INFO - Started process (PID=88) to work on /opt/airflow/dags/script.py
[2024-05-12T10:59:20.636+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/script.py for tasks to queue
[2024-05-12T10:59:20.639+0000] {logging_mixin.py:188} INFO - [2024-05-12T10:59:20.638+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/script.py
[2024-05-12T10:59:20.677+0000] {logging_mixin.py:188} INFO - [2024-05-12T10:59:20.675+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/script.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/script.py", line 19
    def transform_data():
    ^^^
IndentationError: expected an indented block after function definition on line 16
[2024-05-12T10:59:20.678+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/script.py
[2024-05-12T10:59:20.736+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/script.py took 0.109 seconds
[2024-05-12T10:59:51.191+0000] {processor.py:161} INFO - Started process (PID=89) to work on /opt/airflow/dags/script.py
[2024-05-12T10:59:51.193+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/script.py for tasks to queue
[2024-05-12T10:59:51.195+0000] {logging_mixin.py:188} INFO - [2024-05-12T10:59:51.195+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/script.py
[2024-05-12T10:59:51.232+0000] {logging_mixin.py:188} INFO - [2024-05-12T10:59:51.231+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/script.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/script.py", line 19
    def transform_data():
    ^^^
IndentationError: expected an indented block after function definition on line 16
[2024-05-12T10:59:51.234+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/script.py
[2024-05-12T10:59:51.291+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/script.py took 0.107 seconds
[2024-05-12T11:00:21.749+0000] {processor.py:161} INFO - Started process (PID=90) to work on /opt/airflow/dags/script.py
[2024-05-12T11:00:21.750+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/script.py for tasks to queue
[2024-05-12T11:00:21.753+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:00:21.752+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/script.py
[2024-05-12T11:00:21.791+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:00:21.789+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/script.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/script.py", line 19
    def transform_data():
    ^^^
IndentationError: expected an indented block after function definition on line 16
[2024-05-12T11:00:21.793+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/script.py
[2024-05-12T11:00:21.893+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/script.py took 0.152 seconds
[2024-05-12T11:00:52.236+0000] {processor.py:161} INFO - Started process (PID=91) to work on /opt/airflow/dags/script.py
[2024-05-12T11:00:52.237+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/script.py for tasks to queue
[2024-05-12T11:00:52.240+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:00:52.239+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/script.py
[2024-05-12T11:00:52.273+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:00:52.271+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/script.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/script.py", line 19
    def transform_data():
    ^^^
IndentationError: expected an indented block after function definition on line 16
[2024-05-12T11:00:52.274+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/script.py
[2024-05-12T11:00:52.330+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/script.py took 0.100 seconds
[2024-05-12T11:01:22.516+0000] {processor.py:161} INFO - Started process (PID=92) to work on /opt/airflow/dags/script.py
[2024-05-12T11:01:22.517+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/script.py for tasks to queue
[2024-05-12T11:01:22.520+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:01:22.519+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/script.py
[2024-05-12T11:01:22.559+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:01:22.557+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/script.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/script.py", line 19
    def transform_data():
    ^^^
IndentationError: expected an indented block after function definition on line 16
[2024-05-12T11:01:22.568+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/script.py
[2024-05-12T11:01:22.666+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/script.py took 0.157 seconds
[2024-05-12T11:01:53.195+0000] {processor.py:161} INFO - Started process (PID=93) to work on /opt/airflow/dags/script.py
[2024-05-12T11:01:53.197+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/script.py for tasks to queue
[2024-05-12T11:01:53.200+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:01:53.199+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/script.py
[2024-05-12T11:01:53.234+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:01:53.232+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/script.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/script.py", line 19
    def transform_data():
    ^^^
IndentationError: expected an indented block after function definition on line 16
[2024-05-12T11:01:53.235+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/script.py
[2024-05-12T11:01:53.331+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/script.py took 0.143 seconds
[2024-05-12T11:02:23.732+0000] {processor.py:161} INFO - Started process (PID=94) to work on /opt/airflow/dags/script.py
[2024-05-12T11:02:23.733+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/script.py for tasks to queue
[2024-05-12T11:02:23.736+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:02:23.735+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/script.py
[2024-05-12T11:02:23.770+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:02:23.768+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/script.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/script.py", line 19
    def transform_data():
    ^^^
IndentationError: expected an indented block after function definition on line 16
[2024-05-12T11:02:23.771+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/script.py
[2024-05-12T11:02:23.870+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/script.py took 0.146 seconds
[2024-05-12T11:02:54.229+0000] {processor.py:161} INFO - Started process (PID=95) to work on /opt/airflow/dags/script.py
[2024-05-12T11:02:54.231+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/script.py for tasks to queue
[2024-05-12T11:02:54.233+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:02:54.233+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/script.py
[2024-05-12T11:02:54.271+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:02:54.269+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/script.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/script.py", line 19
    def transform_data():
    ^^^
IndentationError: expected an indented block after function definition on line 16
[2024-05-12T11:02:54.272+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/script.py
[2024-05-12T11:02:54.373+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/script.py took 0.151 seconds
[2024-05-12T11:03:24.710+0000] {processor.py:161} INFO - Started process (PID=96) to work on /opt/airflow/dags/script.py
[2024-05-12T11:03:24.711+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/script.py for tasks to queue
[2024-05-12T11:03:24.714+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:03:24.713+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/script.py
[2024-05-12T11:03:24.750+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:03:24.748+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/script.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/script.py", line 19
    def transform_data():
    ^^^
IndentationError: expected an indented block after function definition on line 16
[2024-05-12T11:03:24.751+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/script.py
[2024-05-12T11:03:24.845+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/script.py took 0.141 seconds
[2024-05-12T11:03:55.192+0000] {processor.py:161} INFO - Started process (PID=97) to work on /opt/airflow/dags/script.py
[2024-05-12T11:03:55.216+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/script.py for tasks to queue
[2024-05-12T11:03:55.220+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:03:55.219+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/script.py
[2024-05-12T11:03:55.251+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:03:55.249+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/script.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/script.py", line 19
    def transform_data():
    ^^^
IndentationError: expected an indented block after function definition on line 16
[2024-05-12T11:03:55.252+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/script.py
[2024-05-12T11:03:55.301+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/script.py took 0.116 seconds
[2024-05-12T11:04:25.640+0000] {processor.py:161} INFO - Started process (PID=98) to work on /opt/airflow/dags/script.py
[2024-05-12T11:04:25.642+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/script.py for tasks to queue
[2024-05-12T11:04:25.644+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:04:25.644+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/script.py
[2024-05-12T11:04:25.676+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:04:25.674+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/script.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/script.py", line 19
    def transform_data():
    ^^^
IndentationError: expected an indented block after function definition on line 16
[2024-05-12T11:04:25.677+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/script.py
[2024-05-12T11:04:25.768+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/script.py took 0.135 seconds
[2024-05-12T11:04:56.189+0000] {processor.py:161} INFO - Started process (PID=99) to work on /opt/airflow/dags/script.py
[2024-05-12T11:04:56.191+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/script.py for tasks to queue
[2024-05-12T11:04:56.194+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:04:56.193+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/script.py
[2024-05-12T11:04:56.234+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:04:56.231+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/script.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/script.py", line 19
    def transform_data():
    ^^^
IndentationError: expected an indented block after function definition on line 16
[2024-05-12T11:04:56.235+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/script.py
[2024-05-12T11:04:56.338+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/script.py took 0.156 seconds
[2024-05-12T11:05:26.682+0000] {processor.py:161} INFO - Started process (PID=100) to work on /opt/airflow/dags/script.py
[2024-05-12T11:05:26.684+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/script.py for tasks to queue
[2024-05-12T11:05:26.686+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:05:26.685+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/script.py
[2024-05-12T11:05:26.726+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:05:26.725+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/script.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/script.py", line 19
    def transform_data():
    ^^^
IndentationError: expected an indented block after function definition on line 16
[2024-05-12T11:05:26.728+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/script.py
[2024-05-12T11:05:26.839+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/script.py took 0.164 seconds
[2024-05-12T11:05:57.128+0000] {processor.py:161} INFO - Started process (PID=101) to work on /opt/airflow/dags/script.py
[2024-05-12T11:05:57.130+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/script.py for tasks to queue
[2024-05-12T11:05:57.133+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:05:57.132+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/script.py
[2024-05-12T11:05:57.163+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:05:57.161+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/script.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/script.py", line 19
    def transform_data():
    ^^^
IndentationError: expected an indented block after function definition on line 16
[2024-05-12T11:05:57.164+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/script.py
[2024-05-12T11:05:57.258+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/script.py took 0.137 seconds
[2024-05-12T11:06:27.676+0000] {processor.py:161} INFO - Started process (PID=102) to work on /opt/airflow/dags/script.py
[2024-05-12T11:06:27.677+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/script.py for tasks to queue
[2024-05-12T11:06:27.681+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:06:27.680+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/script.py
[2024-05-12T11:06:27.713+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:06:27.712+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/script.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/script.py", line 19
    def transform_data():
    ^^^
IndentationError: expected an indented block after function definition on line 16
[2024-05-12T11:06:27.714+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/script.py
[2024-05-12T11:06:27.770+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/script.py took 0.100 seconds
[2024-05-12T11:06:57.921+0000] {processor.py:161} INFO - Started process (PID=103) to work on /opt/airflow/dags/script.py
[2024-05-12T11:06:57.922+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/script.py for tasks to queue
[2024-05-12T11:06:57.925+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:06:57.924+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/script.py
[2024-05-12T11:06:57.951+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:06:57.949+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/script.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/script.py", line 19
    def transform_data():
    ^^^
IndentationError: expected an indented block after function definition on line 16
[2024-05-12T11:06:57.952+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/script.py
[2024-05-12T11:06:58.035+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/script.py took 0.121 seconds
[2024-05-12T11:07:28.520+0000] {processor.py:161} INFO - Started process (PID=104) to work on /opt/airflow/dags/script.py
[2024-05-12T11:07:28.522+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/script.py for tasks to queue
[2024-05-12T11:07:28.526+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:07:28.525+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/script.py
[2024-05-12T11:07:28.557+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:07:28.556+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/script.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/script.py", line 19
    def transform_data():
    ^^^
IndentationError: expected an indented block after function definition on line 16
[2024-05-12T11:07:28.559+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/script.py
[2024-05-12T11:07:28.653+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/script.py took 0.140 seconds
[2024-05-12T11:07:59.115+0000] {processor.py:161} INFO - Started process (PID=105) to work on /opt/airflow/dags/script.py
[2024-05-12T11:07:59.117+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/script.py for tasks to queue
[2024-05-12T11:07:59.120+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:07:59.120+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/script.py
[2024-05-12T11:07:59.151+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:07:59.149+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/script.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/script.py", line 19
    def transform_data():
    ^^^
IndentationError: expected an indented block after function definition on line 16
[2024-05-12T11:07:59.153+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/script.py
[2024-05-12T11:07:59.249+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/script.py took 0.141 seconds
[2024-05-12T11:08:29.537+0000] {processor.py:161} INFO - Started process (PID=106) to work on /opt/airflow/dags/script.py
[2024-05-12T11:08:29.539+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/script.py for tasks to queue
[2024-05-12T11:08:29.541+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:08:29.540+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/script.py
[2024-05-12T11:08:29.578+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:08:29.576+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/script.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/script.py", line 19
    def transform_data():
    ^^^
IndentationError: expected an indented block after function definition on line 16
[2024-05-12T11:08:29.579+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/script.py
[2024-05-12T11:08:29.677+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/script.py took 0.147 seconds
[2024-05-12T11:09:00.044+0000] {processor.py:161} INFO - Started process (PID=107) to work on /opt/airflow/dags/script.py
[2024-05-12T11:09:00.045+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/script.py for tasks to queue
[2024-05-12T11:09:00.048+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:09:00.048+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/script.py
[2024-05-12T11:09:00.084+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:09:00.082+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/script.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/script.py", line 19
    def transform_data():
    ^^^
IndentationError: expected an indented block after function definition on line 16
[2024-05-12T11:09:00.085+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/script.py
[2024-05-12T11:09:00.185+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/script.py took 0.149 seconds
[2024-05-12T11:09:30.473+0000] {processor.py:161} INFO - Started process (PID=108) to work on /opt/airflow/dags/script.py
[2024-05-12T11:09:30.475+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/script.py for tasks to queue
[2024-05-12T11:09:30.477+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:09:30.477+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/script.py
[2024-05-12T11:09:30.512+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:09:30.510+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/script.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/script.py", line 19
    def transform_data():
    ^^^
IndentationError: expected an indented block after function definition on line 16
[2024-05-12T11:09:30.513+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/script.py
[2024-05-12T11:09:30.589+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/script.py took 0.123 seconds
[2024-05-12T11:10:01.001+0000] {processor.py:161} INFO - Started process (PID=109) to work on /opt/airflow/dags/script.py
[2024-05-12T11:10:01.002+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/script.py for tasks to queue
[2024-05-12T11:10:01.005+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:10:01.005+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/script.py
[2024-05-12T11:10:01.040+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:10:01.039+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/script.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/script.py", line 19
    def transform_data():
    ^^^
IndentationError: expected an indented block after function definition on line 16
[2024-05-12T11:10:01.042+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/script.py
[2024-05-12T11:10:01.102+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/script.py took 0.110 seconds
[2024-05-12T11:10:31.372+0000] {processor.py:161} INFO - Started process (PID=110) to work on /opt/airflow/dags/script.py
[2024-05-12T11:10:31.374+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/script.py for tasks to queue
[2024-05-12T11:10:31.377+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:10:31.376+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/script.py
[2024-05-12T11:10:31.415+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:10:31.409+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/script.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/script.py", line 19
    def transform_data():
    ^^^
IndentationError: expected an indented block after function definition on line 16
[2024-05-12T11:10:31.417+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/script.py
[2024-05-12T11:10:31.518+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/script.py took 0.152 seconds
[2024-05-12T11:11:01.854+0000] {processor.py:161} INFO - Started process (PID=111) to work on /opt/airflow/dags/script.py
[2024-05-12T11:11:01.856+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/script.py for tasks to queue
[2024-05-12T11:11:01.859+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:11:01.859+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/script.py
[2024-05-12T11:11:01.890+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:11:01.888+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/script.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/script.py", line 19
    def transform_data():
    ^^^
IndentationError: expected an indented block after function definition on line 16
[2024-05-12T11:11:01.891+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/script.py
[2024-05-12T11:11:01.986+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/script.py took 0.142 seconds
[2024-05-12T11:11:32.335+0000] {processor.py:161} INFO - Started process (PID=112) to work on /opt/airflow/dags/script.py
[2024-05-12T11:11:32.336+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/script.py for tasks to queue
[2024-05-12T11:11:32.344+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:11:32.343+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/script.py
[2024-05-12T11:11:32.382+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:11:32.380+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/script.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/script.py", line 19
    def transform_data():
    ^^^
IndentationError: expected an indented block after function definition on line 16
[2024-05-12T11:11:32.383+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/script.py
[2024-05-12T11:11:32.483+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/script.py took 0.158 seconds
[2024-05-12T11:12:02.861+0000] {processor.py:161} INFO - Started process (PID=113) to work on /opt/airflow/dags/script.py
[2024-05-12T11:12:02.862+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/script.py for tasks to queue
[2024-05-12T11:12:02.864+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:12:02.864+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/script.py
[2024-05-12T11:12:02.904+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:12:02.903+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/script.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/script.py", line 19
    def transform_data():
    ^^^
IndentationError: expected an indented block after function definition on line 16
[2024-05-12T11:12:02.906+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/script.py
[2024-05-12T11:12:03.006+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/script.py took 0.152 seconds
[2024-05-12T11:12:33.310+0000] {processor.py:161} INFO - Started process (PID=114) to work on /opt/airflow/dags/script.py
[2024-05-12T11:12:33.312+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/script.py for tasks to queue
[2024-05-12T11:12:33.316+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:12:33.316+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/script.py
[2024-05-12T11:12:33.355+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:12:33.354+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/script.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/script.py", line 19
    def transform_data():
    ^^^
IndentationError: expected an indented block after function definition on line 16
[2024-05-12T11:12:33.356+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/script.py
[2024-05-12T11:12:33.458+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/script.py took 0.162 seconds
[2024-05-12T11:13:03.886+0000] {processor.py:161} INFO - Started process (PID=115) to work on /opt/airflow/dags/script.py
[2024-05-12T11:13:03.887+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/script.py for tasks to queue
[2024-05-12T11:13:03.891+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:13:03.891+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/script.py
[2024-05-12T11:13:03.930+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:13:03.928+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/script.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/script.py", line 19
    def transform_data():
    ^^^
IndentationError: expected an indented block after function definition on line 16
[2024-05-12T11:13:03.931+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/script.py
[2024-05-12T11:13:04.032+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/script.py took 0.154 seconds
[2024-05-12T11:13:34.358+0000] {processor.py:161} INFO - Started process (PID=116) to work on /opt/airflow/dags/script.py
[2024-05-12T11:13:34.360+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/script.py for tasks to queue
[2024-05-12T11:13:34.362+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:13:34.362+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/script.py
[2024-05-12T11:13:34.398+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:13:34.396+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/script.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/script.py", line 19
    def transform_data():
    ^^^
IndentationError: expected an indented block after function definition on line 16
[2024-05-12T11:13:34.399+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/script.py
[2024-05-12T11:13:34.453+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/script.py took 0.104 seconds
[2024-05-12T11:14:04.868+0000] {processor.py:161} INFO - Started process (PID=117) to work on /opt/airflow/dags/script.py
[2024-05-12T11:14:04.869+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/script.py for tasks to queue
[2024-05-12T11:14:04.872+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:14:04.871+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/script.py
[2024-05-12T11:14:04.912+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:14:04.910+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/script.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/script.py", line 19
    def transform_data():
    ^^^
IndentationError: expected an indented block after function definition on line 16
[2024-05-12T11:14:04.913+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/script.py
[2024-05-12T11:14:05.013+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/script.py took 0.154 seconds
[2024-05-12T11:14:35.211+0000] {processor.py:161} INFO - Started process (PID=118) to work on /opt/airflow/dags/script.py
[2024-05-12T11:14:35.213+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/script.py for tasks to queue
[2024-05-12T11:14:35.216+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:14:35.215+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/script.py
[2024-05-12T11:14:35.255+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:14:35.253+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/script.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/script.py", line 19
    def transform_data():
    ^^^
IndentationError: expected an indented block after function definition on line 16
[2024-05-12T11:14:35.256+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/script.py
[2024-05-12T11:14:35.357+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/script.py took 0.153 seconds
[2024-05-12T11:15:05.790+0000] {processor.py:161} INFO - Started process (PID=119) to work on /opt/airflow/dags/script.py
[2024-05-12T11:15:05.792+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/script.py for tasks to queue
[2024-05-12T11:15:05.794+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:15:05.794+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/script.py
[2024-05-12T11:15:05.833+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:15:05.831+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/script.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/script.py", line 19
    def transform_data():
    ^^^
IndentationError: expected an indented block after function definition on line 16
[2024-05-12T11:15:05.834+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/script.py
[2024-05-12T11:15:05.931+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/script.py took 0.148 seconds
[2024-05-12T11:15:36.256+0000] {processor.py:161} INFO - Started process (PID=120) to work on /opt/airflow/dags/script.py
[2024-05-12T11:15:36.257+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/script.py for tasks to queue
[2024-05-12T11:15:36.260+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:15:36.259+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/script.py
[2024-05-12T11:15:36.296+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:15:36.294+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/script.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/script.py", line 19
    def transform_data():
    ^^^
IndentationError: expected an indented block after function definition on line 16
[2024-05-12T11:15:36.297+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/script.py
[2024-05-12T11:15:36.360+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/script.py took 0.110 seconds
[2024-05-12T11:16:06.818+0000] {processor.py:161} INFO - Started process (PID=121) to work on /opt/airflow/dags/script.py
[2024-05-12T11:16:06.819+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/script.py for tasks to queue
[2024-05-12T11:16:06.822+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:16:06.821+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/script.py
[2024-05-12T11:16:06.859+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:16:06.857+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/script.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/script.py", line 19
    def transform_data():
    ^^^
IndentationError: expected an indented block after function definition on line 16
[2024-05-12T11:16:06.860+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/script.py
[2024-05-12T11:16:06.923+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/script.py took 0.111 seconds
[2024-05-12T11:16:37.218+0000] {processor.py:161} INFO - Started process (PID=122) to work on /opt/airflow/dags/script.py
[2024-05-12T11:16:37.220+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/script.py for tasks to queue
[2024-05-12T11:16:37.223+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:16:37.222+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/script.py
[2024-05-12T11:16:37.262+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:16:37.260+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/script.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/script.py", line 19
    def transform_data():
    ^^^
IndentationError: expected an indented block after function definition on line 16
[2024-05-12T11:16:37.263+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/script.py
[2024-05-12T11:16:37.367+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/script.py took 0.155 seconds
[2024-05-12T11:17:07.718+0000] {processor.py:161} INFO - Started process (PID=123) to work on /opt/airflow/dags/script.py
[2024-05-12T11:17:07.722+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/script.py for tasks to queue
[2024-05-12T11:17:07.730+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:17:07.729+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/script.py
[2024-05-12T11:17:07.764+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:17:07.761+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/script.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/script.py", line 19
    def transform_data():
    ^^^
IndentationError: expected an indented block after function definition on line 16
[2024-05-12T11:17:07.766+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/script.py
[2024-05-12T11:17:07.844+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/script.py took 0.136 seconds
[2024-05-12T11:17:38.096+0000] {processor.py:161} INFO - Started process (PID=124) to work on /opt/airflow/dags/script.py
[2024-05-12T11:17:38.098+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/script.py for tasks to queue
[2024-05-12T11:17:38.101+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:17:38.100+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/script.py
[2024-05-12T11:17:38.137+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:17:38.135+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/script.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/script.py", line 19
    def transform_data():
    ^^^
IndentationError: expected an indented block after function definition on line 16
[2024-05-12T11:17:38.138+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/script.py
[2024-05-12T11:17:38.243+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/script.py took 0.157 seconds
[2024-05-12T11:18:08.624+0000] {processor.py:161} INFO - Started process (PID=125) to work on /opt/airflow/dags/script.py
[2024-05-12T11:18:08.625+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/script.py for tasks to queue
[2024-05-12T11:18:08.629+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:18:08.628+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/script.py
[2024-05-12T11:18:08.665+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:18:08.662+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/script.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/script.py", line 19
    def transform_data():
    ^^^
IndentationError: expected an indented block after function definition on line 16
[2024-05-12T11:18:08.667+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/script.py
[2024-05-12T11:18:08.732+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/script.py took 0.116 seconds
[2024-05-12T11:18:39.211+0000] {processor.py:161} INFO - Started process (PID=126) to work on /opt/airflow/dags/script.py
[2024-05-12T11:18:39.213+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/script.py for tasks to queue
[2024-05-12T11:18:39.215+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:18:39.215+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/script.py
[2024-05-12T11:18:39.253+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:18:39.251+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/script.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/script.py", line 19
    def transform_data():
    ^^^
IndentationError: expected an indented block after function definition on line 16
[2024-05-12T11:18:39.254+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/script.py
[2024-05-12T11:18:39.316+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/script.py took 0.113 seconds
[2024-05-12T11:19:09.764+0000] {processor.py:161} INFO - Started process (PID=127) to work on /opt/airflow/dags/script.py
[2024-05-12T11:19:09.765+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/script.py for tasks to queue
[2024-05-12T11:19:09.768+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:19:09.768+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/script.py
[2024-05-12T11:19:09.806+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:19:09.804+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/script.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/script.py", line 19
    def transform_data():
    ^^^
IndentationError: expected an indented block after function definition on line 16
[2024-05-12T11:19:09.816+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/script.py
[2024-05-12T11:19:09.878+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/script.py took 0.121 seconds
[2024-05-12T11:19:40.324+0000] {processor.py:161} INFO - Started process (PID=128) to work on /opt/airflow/dags/script.py
[2024-05-12T11:19:40.325+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/script.py for tasks to queue
[2024-05-12T11:19:40.328+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:19:40.328+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/script.py
[2024-05-12T11:19:40.365+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:19:40.364+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/script.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/script.py", line 19
    def transform_data():
    ^^^
IndentationError: expected an indented block after function definition on line 16
[2024-05-12T11:19:40.367+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/script.py
[2024-05-12T11:19:40.428+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/script.py took 0.114 seconds
[2024-05-12T11:20:10.935+0000] {processor.py:161} INFO - Started process (PID=129) to work on /opt/airflow/dags/script.py
[2024-05-12T11:20:10.937+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/script.py for tasks to queue
[2024-05-12T11:20:10.939+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:20:10.939+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/script.py
[2024-05-12T11:20:10.967+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:20:10.965+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/script.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/script.py", line 19
    def transform_data():
    ^^^
IndentationError: expected an indented block after function definition on line 16
[2024-05-12T11:20:10.968+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/script.py
[2024-05-12T11:20:11.060+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/script.py took 0.131 seconds
[2024-05-12T11:20:41.487+0000] {processor.py:161} INFO - Started process (PID=130) to work on /opt/airflow/dags/script.py
[2024-05-12T11:20:41.489+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/script.py for tasks to queue
[2024-05-12T11:20:41.491+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:20:41.491+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/script.py
[2024-05-12T11:20:41.522+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:20:41.520+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/script.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/script.py", line 19
    def transform_data():
    ^^^
IndentationError: expected an indented block after function definition on line 16
[2024-05-12T11:20:41.523+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/script.py
[2024-05-12T11:20:41.575+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/script.py took 0.099 seconds
[2024-05-12T11:21:12.091+0000] {processor.py:161} INFO - Started process (PID=131) to work on /opt/airflow/dags/script.py
[2024-05-12T11:21:12.093+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/script.py for tasks to queue
[2024-05-12T11:21:12.096+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:21:12.095+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/script.py
[2024-05-12T11:21:12.135+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:21:12.133+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/script.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/script.py", line 19
    def transform_data():
    ^^^
IndentationError: expected an indented block after function definition on line 16
[2024-05-12T11:21:12.136+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/script.py
[2024-05-12T11:21:12.196+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/script.py took 0.112 seconds
[2024-05-12T11:21:42.663+0000] {processor.py:161} INFO - Started process (PID=132) to work on /opt/airflow/dags/script.py
[2024-05-12T11:21:42.664+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/script.py for tasks to queue
[2024-05-12T11:21:42.667+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:21:42.666+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/script.py
[2024-05-12T11:21:42.703+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:21:42.702+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/script.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/script.py", line 19
    def transform_data():
    ^^^
IndentationError: expected an indented block after function definition on line 16
[2024-05-12T11:21:42.705+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/script.py
[2024-05-12T11:21:42.765+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/script.py took 0.110 seconds
[2024-05-12T11:22:13.202+0000] {processor.py:161} INFO - Started process (PID=133) to work on /opt/airflow/dags/script.py
[2024-05-12T11:22:13.203+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/script.py for tasks to queue
[2024-05-12T11:22:13.205+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:22:13.205+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/script.py
[2024-05-12T11:22:13.230+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:22:13.229+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/script.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/script.py", line 19
    def transform_data():
    ^^^
IndentationError: expected an indented block after function definition on line 16
[2024-05-12T11:22:13.231+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/script.py
[2024-05-12T11:22:13.328+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/script.py took 0.136 seconds
[2024-05-12T11:22:43.702+0000] {processor.py:161} INFO - Started process (PID=134) to work on /opt/airflow/dags/script.py
[2024-05-12T11:22:43.703+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/script.py for tasks to queue
[2024-05-12T11:22:43.705+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:22:43.705+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/script.py
[2024-05-12T11:22:43.731+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:22:43.730+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/script.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/script.py", line 19
    def transform_data():
    ^^^
IndentationError: expected an indented block after function definition on line 16
[2024-05-12T11:22:43.732+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/script.py
[2024-05-12T11:22:43.817+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/script.py took 0.122 seconds
[2024-05-12T11:23:14.023+0000] {processor.py:161} INFO - Started process (PID=135) to work on /opt/airflow/dags/script.py
[2024-05-12T11:23:14.024+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/script.py for tasks to queue
[2024-05-12T11:23:14.026+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:23:14.025+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/script.py
[2024-05-12T11:23:14.049+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:23:14.047+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/script.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/script.py", line 19
    def transform_data():
    ^^^
IndentationError: expected an indented block after function definition on line 16
[2024-05-12T11:23:14.049+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/script.py
[2024-05-12T11:23:14.093+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/script.py took 0.076 seconds
[2024-05-12T11:23:44.456+0000] {processor.py:161} INFO - Started process (PID=136) to work on /opt/airflow/dags/script.py
[2024-05-12T11:23:44.457+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/script.py for tasks to queue
[2024-05-12T11:23:44.460+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:23:44.460+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/script.py
[2024-05-12T11:23:44.484+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:23:44.482+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/script.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/script.py", line 19
    def transform_data():
    ^^^
IndentationError: expected an indented block after function definition on line 16
[2024-05-12T11:23:44.485+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/script.py
[2024-05-12T11:23:44.528+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/script.py took 0.077 seconds
[2024-05-12T11:24:14.834+0000] {processor.py:161} INFO - Started process (PID=137) to work on /opt/airflow/dags/script.py
[2024-05-12T11:24:14.836+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/script.py for tasks to queue
[2024-05-12T11:24:14.838+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:24:14.838+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/script.py
[2024-05-12T11:24:14.866+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:24:14.865+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/script.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/script.py", line 19
    def transform_data():
    ^^^
IndentationError: expected an indented block after function definition on line 16
[2024-05-12T11:24:14.867+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/script.py
[2024-05-12T11:24:14.915+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/script.py took 0.086 seconds
[2024-05-12T11:24:45.336+0000] {processor.py:161} INFO - Started process (PID=138) to work on /opt/airflow/dags/script.py
[2024-05-12T11:24:45.338+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/script.py for tasks to queue
[2024-05-12T11:24:45.347+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:24:45.340+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/script.py
[2024-05-12T11:24:45.372+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:24:45.370+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/script.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/script.py", line 19
    def transform_data():
    ^^^
IndentationError: expected an indented block after function definition on line 16
[2024-05-12T11:24:45.373+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/script.py
[2024-05-12T11:24:45.461+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/script.py took 0.131 seconds
[2024-05-12T11:25:15.801+0000] {processor.py:161} INFO - Started process (PID=139) to work on /opt/airflow/dags/script.py
[2024-05-12T11:25:15.802+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/script.py for tasks to queue
[2024-05-12T11:25:15.805+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:25:15.804+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/script.py
[2024-05-12T11:25:15.828+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:25:15.827+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/script.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/script.py", line 19
    def transform_data():
    ^^^
IndentationError: expected an indented block after function definition on line 16
[2024-05-12T11:25:15.829+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/script.py
[2024-05-12T11:25:15.915+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/script.py took 0.120 seconds
[2024-05-12T11:25:46.251+0000] {processor.py:161} INFO - Started process (PID=140) to work on /opt/airflow/dags/script.py
[2024-05-12T11:25:46.252+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/script.py for tasks to queue
[2024-05-12T11:25:46.254+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:25:46.254+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/script.py
[2024-05-12T11:25:46.278+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:25:46.276+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/script.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/script.py", line 19
    def transform_data():
    ^^^
IndentationError: expected an indented block after function definition on line 16
[2024-05-12T11:25:46.278+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/script.py
[2024-05-12T11:25:46.325+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/script.py took 0.079 seconds
[2024-05-12T11:26:16.593+0000] {processor.py:161} INFO - Started process (PID=141) to work on /opt/airflow/dags/script.py
[2024-05-12T11:26:16.594+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/script.py for tasks to queue
[2024-05-12T11:26:16.596+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:26:16.596+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/script.py
[2024-05-12T11:26:16.619+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:26:16.617+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/script.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/script.py", line 19
    def transform_data():
    ^^^
IndentationError: expected an indented block after function definition on line 16
[2024-05-12T11:26:16.620+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/script.py
[2024-05-12T11:26:16.663+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/script.py took 0.077 seconds
[2024-05-12T11:26:46.945+0000] {processor.py:161} INFO - Started process (PID=142) to work on /opt/airflow/dags/script.py
[2024-05-12T11:26:46.947+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/script.py for tasks to queue
[2024-05-12T11:26:46.949+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:26:46.949+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/script.py
[2024-05-12T11:26:46.989+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:26:46.987+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/script.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/script.py", line 19
    def transform_data():
    ^^^
IndentationError: expected an indented block after function definition on line 16
[2024-05-12T11:26:46.990+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/script.py
[2024-05-12T11:26:47.053+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/script.py took 0.116 seconds
[2024-05-12T11:27:17.323+0000] {processor.py:161} INFO - Started process (PID=143) to work on /opt/airflow/dags/script.py
[2024-05-12T11:27:17.324+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/script.py for tasks to queue
[2024-05-12T11:27:17.327+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:27:17.326+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/script.py
[2024-05-12T11:27:17.361+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:27:17.360+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/script.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/script.py", line 19
    def transform_data():
    ^^^
IndentationError: expected an indented block after function definition on line 16
[2024-05-12T11:27:17.362+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/script.py
[2024-05-12T11:27:17.413+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/script.py took 0.095 seconds
[2024-05-12T11:27:47.748+0000] {processor.py:161} INFO - Started process (PID=144) to work on /opt/airflow/dags/script.py
[2024-05-12T11:27:47.749+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/script.py for tasks to queue
[2024-05-12T11:27:47.751+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:27:47.751+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/script.py
[2024-05-12T11:27:47.772+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:27:47.771+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/script.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/script.py", line 19
    def transform_data():
    ^^^
IndentationError: expected an indented block after function definition on line 16
[2024-05-12T11:27:47.773+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/script.py
[2024-05-12T11:27:47.855+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/script.py took 0.112 seconds
[2024-05-12T11:28:18.209+0000] {processor.py:161} INFO - Started process (PID=145) to work on /opt/airflow/dags/script.py
[2024-05-12T11:28:18.210+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/script.py for tasks to queue
[2024-05-12T11:28:18.212+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:28:18.212+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/script.py
[2024-05-12T11:28:18.237+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:28:18.235+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/script.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/script.py", line 19
    def transform_data():
    ^^^
IndentationError: expected an indented block after function definition on line 16
[2024-05-12T11:28:18.239+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/script.py
[2024-05-12T11:28:18.286+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/script.py took 0.085 seconds
[2024-05-12T11:28:48.649+0000] {processor.py:161} INFO - Started process (PID=146) to work on /opt/airflow/dags/script.py
[2024-05-12T11:28:48.650+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/script.py for tasks to queue
[2024-05-12T11:28:48.653+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:28:48.652+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/script.py
[2024-05-12T11:28:48.687+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:28:48.686+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/script.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/script.py", line 19
    def transform_data():
    ^^^
IndentationError: expected an indented block after function definition on line 16
[2024-05-12T11:28:48.689+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/script.py
[2024-05-12T11:28:48.754+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/script.py took 0.112 seconds
[2024-05-12T11:29:19.207+0000] {processor.py:161} INFO - Started process (PID=147) to work on /opt/airflow/dags/script.py
[2024-05-12T11:29:19.209+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/script.py for tasks to queue
[2024-05-12T11:29:19.212+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:29:19.212+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/script.py
[2024-05-12T11:29:19.252+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:29:19.250+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/script.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/script.py", line 19
    def transform_data():
    ^^^
IndentationError: expected an indented block after function definition on line 16
[2024-05-12T11:29:19.253+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/script.py
[2024-05-12T11:29:19.360+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/script.py took 0.163 seconds
[2024-05-12T11:29:49.755+0000] {processor.py:161} INFO - Started process (PID=148) to work on /opt/airflow/dags/script.py
[2024-05-12T11:29:49.762+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/script.py for tasks to queue
[2024-05-12T11:29:49.765+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:29:49.765+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/script.py
[2024-05-12T11:29:49.790+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:29:49.789+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/script.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/script.py", line 19
    def transform_data():
    ^^^
IndentationError: expected an indented block after function definition on line 16
[2024-05-12T11:29:49.791+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/script.py
[2024-05-12T11:29:49.873+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/script.py took 0.123 seconds
[2024-05-12T11:30:20.292+0000] {processor.py:161} INFO - Started process (PID=149) to work on /opt/airflow/dags/script.py
[2024-05-12T11:30:20.294+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/script.py for tasks to queue
[2024-05-12T11:30:20.297+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:30:20.297+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/script.py
[2024-05-12T11:30:20.336+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:30:20.334+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/script.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/script.py", line 19
    def transform_data():
    ^^^
IndentationError: expected an indented block after function definition on line 16
[2024-05-12T11:30:20.338+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/script.py
[2024-05-12T11:30:20.441+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/script.py took 0.156 seconds
[2024-05-12T11:30:50.847+0000] {processor.py:161} INFO - Started process (PID=150) to work on /opt/airflow/dags/script.py
[2024-05-12T11:30:50.849+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/script.py for tasks to queue
[2024-05-12T11:30:50.852+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:30:50.851+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/script.py
[2024-05-12T11:30:50.885+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:30:50.883+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/script.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/script.py", line 19
    def transform_data():
    ^^^
IndentationError: expected an indented block after function definition on line 16
[2024-05-12T11:30:50.919+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/script.py
[2024-05-12T11:30:50.998+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/script.py took 0.159 seconds
[2024-05-12T11:31:21.373+0000] {processor.py:161} INFO - Started process (PID=151) to work on /opt/airflow/dags/script.py
[2024-05-12T11:31:21.374+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/script.py for tasks to queue
[2024-05-12T11:31:21.378+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:31:21.377+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/script.py
[2024-05-12T11:31:21.418+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:31:21.417+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/script.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/script.py", line 19
    def transform_data():
    ^^^
IndentationError: expected an indented block after function definition on line 16
[2024-05-12T11:31:21.419+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/script.py
[2024-05-12T11:31:21.517+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/script.py took 0.151 seconds
[2024-05-12T11:31:51.869+0000] {processor.py:161} INFO - Started process (PID=152) to work on /opt/airflow/dags/script.py
[2024-05-12T11:31:51.870+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/script.py for tasks to queue
[2024-05-12T11:31:51.872+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:31:51.872+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/script.py
[2024-05-12T11:31:51.899+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:31:51.897+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/script.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/script.py", line 19
    def transform_data():
    ^^^
IndentationError: expected an indented block after function definition on line 16
[2024-05-12T11:31:51.899+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/script.py
[2024-05-12T11:31:51.985+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/script.py took 0.122 seconds
[2024-05-12T11:32:22.325+0000] {processor.py:161} INFO - Started process (PID=153) to work on /opt/airflow/dags/script.py
[2024-05-12T11:32:22.327+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/script.py for tasks to queue
[2024-05-12T11:32:22.329+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:32:22.328+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/script.py
[2024-05-12T11:32:22.353+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:32:22.351+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/script.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/script.py", line 19
    def transform_data():
    ^^^
IndentationError: expected an indented block after function definition on line 16
[2024-05-12T11:32:22.354+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/script.py
[2024-05-12T11:32:22.399+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/script.py took 0.079 seconds
[2024-05-12T11:32:52.699+0000] {processor.py:161} INFO - Started process (PID=154) to work on /opt/airflow/dags/script.py
[2024-05-12T11:32:52.701+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/script.py for tasks to queue
[2024-05-12T11:32:52.702+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:32:52.702+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/script.py
[2024-05-12T11:32:52.728+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:32:52.727+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/script.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/script.py", line 19
    def transform_data():
    ^^^
IndentationError: expected an indented block after function definition on line 16
[2024-05-12T11:32:52.729+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/script.py
[2024-05-12T11:32:52.814+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/script.py took 0.120 seconds
[2024-05-12T11:33:23.269+0000] {processor.py:161} INFO - Started process (PID=155) to work on /opt/airflow/dags/script.py
[2024-05-12T11:33:23.274+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/script.py for tasks to queue
[2024-05-12T11:33:23.278+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:33:23.277+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/script.py
[2024-05-12T11:33:23.313+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:33:23.311+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/script.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/script.py", line 19
    def transform_data():
    ^^^
IndentationError: expected an indented block after function definition on line 16
[2024-05-12T11:33:23.315+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/script.py
[2024-05-12T11:33:23.403+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/script.py took 0.148 seconds
[2024-05-12T11:33:53.816+0000] {processor.py:161} INFO - Started process (PID=156) to work on /opt/airflow/dags/script.py
[2024-05-12T11:33:53.818+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/script.py for tasks to queue
[2024-05-12T11:33:53.822+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:33:53.821+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/script.py
[2024-05-12T11:33:53.845+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:33:53.843+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/script.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/script.py", line 19
    def transform_data():
    ^^^
IndentationError: expected an indented block after function definition on line 16
[2024-05-12T11:33:53.846+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/script.py
[2024-05-12T11:33:53.910+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/script.py took 0.114 seconds
[2024-05-12T11:34:25.408+0000] {processor.py:161} INFO - Started process (PID=157) to work on /opt/airflow/dags/script.py
[2024-05-12T11:34:25.409+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/script.py for tasks to queue
[2024-05-12T11:34:25.413+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:34:25.412+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/script.py
[2024-05-12T11:34:25.451+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:34:25.447+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/script.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/script.py", line 19
    def transform_data():
    ^^^
IndentationError: expected an indented block after function definition on line 16
[2024-05-12T11:34:25.453+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/script.py
[2024-05-12T11:34:25.574+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/script.py took 0.177 seconds
[2024-05-12T11:34:55.783+0000] {processor.py:161} INFO - Started process (PID=158) to work on /opt/airflow/dags/script.py
[2024-05-12T11:34:55.785+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/script.py for tasks to queue
[2024-05-12T11:34:55.789+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:34:55.788+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/script.py
[2024-05-12T11:34:55.832+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:34:55.830+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/script.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/script.py", line 19
    def transform_data():
    ^^^
IndentationError: expected an indented block after function definition on line 16
[2024-05-12T11:34:55.833+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/script.py
[2024-05-12T11:34:55.908+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/script.py took 0.132 seconds
[2024-05-12T11:35:26.082+0000] {processor.py:161} INFO - Started process (PID=159) to work on /opt/airflow/dags/script.py
[2024-05-12T11:35:26.083+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/script.py for tasks to queue
[2024-05-12T11:35:26.086+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:35:26.086+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/script.py
[2024-05-12T11:35:26.122+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:35:26.120+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/script.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/script.py", line 19
    def transform_data():
    ^^^
IndentationError: expected an indented block after function definition on line 16
[2024-05-12T11:35:26.123+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/script.py
[2024-05-12T11:35:26.223+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/script.py took 0.151 seconds
[2024-05-12T11:35:56.343+0000] {processor.py:161} INFO - Started process (PID=160) to work on /opt/airflow/dags/script.py
[2024-05-12T11:35:56.355+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/script.py for tasks to queue
[2024-05-12T11:35:56.361+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:35:56.361+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/script.py
[2024-05-12T11:35:56.398+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:35:56.396+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/script.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/script.py", line 19
    def transform_data():
    ^^^
IndentationError: expected an indented block after function definition on line 16
[2024-05-12T11:35:56.399+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/script.py
[2024-05-12T11:35:56.461+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/script.py took 0.138 seconds
[2024-05-12T11:36:26.662+0000] {processor.py:161} INFO - Started process (PID=161) to work on /opt/airflow/dags/script.py
[2024-05-12T11:36:26.663+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/script.py for tasks to queue
[2024-05-12T11:36:26.666+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:36:26.665+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/script.py
[2024-05-12T11:36:26.700+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:36:26.698+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/script.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/script.py", line 19
    def transform_data():
    ^^^
IndentationError: expected an indented block after function definition on line 16
[2024-05-12T11:36:26.701+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/script.py
[2024-05-12T11:36:26.805+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/script.py took 0.149 seconds
[2024-05-12T11:36:56.962+0000] {processor.py:161} INFO - Started process (PID=162) to work on /opt/airflow/dags/script.py
[2024-05-12T11:36:56.964+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/script.py for tasks to queue
[2024-05-12T11:36:56.967+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:36:56.967+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/script.py
[2024-05-12T11:36:57.008+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:36:57.004+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/script.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/script.py", line 19
    def transform_data():
    ^^^
IndentationError: expected an indented block after function definition on line 16
[2024-05-12T11:36:57.009+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/script.py
[2024-05-12T11:36:57.116+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/script.py took 0.161 seconds
[2024-05-12T11:37:04.038+0000] {processor.py:161} INFO - Started process (PID=163) to work on /opt/airflow/dags/script.py
[2024-05-12T11:37:04.040+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/script.py for tasks to queue
[2024-05-12T11:37:04.043+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:37:04.042+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/script.py
[2024-05-12T11:37:04.192+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:37:04.184+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/script.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/script.py", line 4, in <module>
    from pydrive.auth import GoogleAuth
ModuleNotFoundError: No module named 'pydrive'
[2024-05-12T11:37:04.196+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/script.py
[2024-05-12T11:37:04.256+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/script.py took 0.227 seconds
[2024-05-12T11:37:34.513+0000] {processor.py:161} INFO - Started process (PID=164) to work on /opt/airflow/dags/script.py
[2024-05-12T11:37:34.516+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/script.py for tasks to queue
[2024-05-12T11:37:34.518+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:37:34.517+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/script.py
[2024-05-12T11:37:34.572+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:37:34.562+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/script.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/script.py", line 4, in <module>
    from pydrive.auth import GoogleAuth
ModuleNotFoundError: No module named 'pydrive'
[2024-05-12T11:37:34.576+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/script.py
[2024-05-12T11:37:34.689+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/script.py took 0.194 seconds
[2024-05-12T11:38:04.990+0000] {processor.py:161} INFO - Started process (PID=165) to work on /opt/airflow/dags/script.py
[2024-05-12T11:38:04.992+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/script.py for tasks to queue
[2024-05-12T11:38:04.994+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:38:04.994+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/script.py
[2024-05-12T11:38:05.034+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:38:05.028+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/script.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/script.py", line 4, in <module>
    from pydrive.auth import GoogleAuth
ModuleNotFoundError: No module named 'pydrive'
[2024-05-12T11:38:05.039+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/script.py
[2024-05-12T11:38:05.137+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/script.py took 0.154 seconds
[2024-05-12T11:38:35.866+0000] {processor.py:161} INFO - Started process (PID=166) to work on /opt/airflow/dags/script.py
[2024-05-12T11:38:35.867+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/script.py for tasks to queue
[2024-05-12T11:38:35.870+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:38:35.869+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/script.py
[2024-05-12T11:38:35.914+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:38:35.905+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/script.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/script.py", line 4, in <module>
    from pydrive.auth import GoogleAuth
ModuleNotFoundError: No module named 'pydrive'
[2024-05-12T11:38:35.918+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/script.py
[2024-05-12T11:38:36.025+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/script.py took 0.165 seconds
[2024-05-12T11:39:07.222+0000] {processor.py:161} INFO - Started process (PID=167) to work on /opt/airflow/dags/script.py
[2024-05-12T11:39:07.223+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/script.py for tasks to queue
[2024-05-12T11:39:07.226+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:39:07.226+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/script.py
[2024-05-12T11:39:07.279+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:39:07.268+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/script.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/script.py", line 4, in <module>
    from pydrive.auth import GoogleAuth
ModuleNotFoundError: No module named 'pydrive'
[2024-05-12T11:39:07.283+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/script.py
[2024-05-12T11:39:07.395+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/script.py took 0.179 seconds
[2024-05-12T11:39:38.664+0000] {processor.py:161} INFO - Started process (PID=168) to work on /opt/airflow/dags/script.py
[2024-05-12T11:39:38.665+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/script.py for tasks to queue
[2024-05-12T11:39:38.668+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:39:38.668+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/script.py
[2024-05-12T11:39:38.709+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:39:38.701+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/script.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/script.py", line 4, in <module>
    from pydrive.auth import GoogleAuth
ModuleNotFoundError: No module named 'pydrive'
[2024-05-12T11:39:38.713+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/script.py
[2024-05-12T11:39:38.814+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/script.py took 0.157 seconds
[2024-05-12T11:40:09.850+0000] {processor.py:161} INFO - Started process (PID=169) to work on /opt/airflow/dags/script.py
[2024-05-12T11:40:09.851+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/script.py for tasks to queue
[2024-05-12T11:40:09.853+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:40:09.852+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/script.py
[2024-05-12T11:40:09.872+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:40:09.868+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/script.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/script.py", line 4, in <module>
    from pydrive.auth import GoogleAuth
ModuleNotFoundError: No module named 'pydrive'
[2024-05-12T11:40:09.873+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/script.py
[2024-05-12T11:40:09.899+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/script.py took 0.053 seconds
[2024-05-12T11:40:40.466+0000] {processor.py:161} INFO - Started process (PID=170) to work on /opt/airflow/dags/script.py
[2024-05-12T11:40:40.467+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/script.py for tasks to queue
[2024-05-12T11:40:40.468+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:40:40.468+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/script.py
[2024-05-12T11:40:40.488+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:40:40.484+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/script.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/script.py", line 4, in <module>
    from pydrive.auth import GoogleAuth
ModuleNotFoundError: No module named 'pydrive'
[2024-05-12T11:40:40.489+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/script.py
[2024-05-12T11:40:40.516+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/script.py took 0.055 seconds
[2024-05-12T11:41:11.068+0000] {processor.py:161} INFO - Started process (PID=171) to work on /opt/airflow/dags/script.py
[2024-05-12T11:41:11.069+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/script.py for tasks to queue
[2024-05-12T11:41:11.071+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:41:11.070+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/script.py
[2024-05-12T11:41:11.086+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:41:11.083+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/script.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/script.py", line 4, in <module>
    from pydrive.auth import GoogleAuth
ModuleNotFoundError: No module named 'pydrive'
[2024-05-12T11:41:11.087+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/script.py
[2024-05-12T11:41:11.121+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/script.py took 0.057 seconds
[2024-05-12T11:41:41.653+0000] {processor.py:161} INFO - Started process (PID=172) to work on /opt/airflow/dags/script.py
[2024-05-12T11:41:41.654+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/script.py for tasks to queue
[2024-05-12T11:41:41.656+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:41:41.656+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/script.py
[2024-05-12T11:41:41.675+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:41:41.671+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/script.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/script.py", line 4, in <module>
    from pydrive.auth import GoogleAuth
ModuleNotFoundError: No module named 'pydrive'
[2024-05-12T11:41:41.676+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/script.py
[2024-05-12T11:41:41.721+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/script.py took 0.072 seconds
[2024-05-12T11:42:12.203+0000] {processor.py:161} INFO - Started process (PID=173) to work on /opt/airflow/dags/script.py
[2024-05-12T11:42:12.204+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/script.py for tasks to queue
[2024-05-12T11:42:12.205+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:42:12.205+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/script.py
[2024-05-12T11:42:12.220+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:42:12.217+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/script.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/script.py", line 4, in <module>
    from pydrive.auth import GoogleAuth
ModuleNotFoundError: No module named 'pydrive'
[2024-05-12T11:42:12.221+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/script.py
[2024-05-12T11:42:12.256+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/script.py took 0.057 seconds
[2024-05-12T11:42:42.770+0000] {processor.py:161} INFO - Started process (PID=174) to work on /opt/airflow/dags/script.py
[2024-05-12T11:42:42.771+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/script.py for tasks to queue
[2024-05-12T11:42:42.773+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:42:42.772+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/script.py
[2024-05-12T11:42:42.790+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:42:42.787+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/script.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/script.py", line 4, in <module>
    from pydrive.auth import GoogleAuth
ModuleNotFoundError: No module named 'pydrive'
[2024-05-12T11:42:42.792+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/script.py
[2024-05-12T11:42:42.829+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/script.py took 0.062 seconds
[2024-05-12T11:43:13.189+0000] {processor.py:161} INFO - Started process (PID=175) to work on /opt/airflow/dags/script.py
[2024-05-12T11:43:13.190+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/script.py for tasks to queue
[2024-05-12T11:43:13.192+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:43:13.191+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/script.py
[2024-05-12T11:43:13.209+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:43:13.206+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/script.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/script.py", line 4, in <module>
    from pydrive.auth import GoogleAuth
ModuleNotFoundError: No module named 'pydrive'
[2024-05-12T11:43:13.210+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/script.py
[2024-05-12T11:43:13.235+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/script.py took 0.049 seconds
[2024-05-12T11:43:43.529+0000] {processor.py:161} INFO - Started process (PID=176) to work on /opt/airflow/dags/script.py
[2024-05-12T11:43:43.530+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/script.py for tasks to queue
[2024-05-12T11:43:43.532+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:43:43.531+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/script.py
[2024-05-12T11:43:43.550+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:43:43.546+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/script.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/script.py", line 4, in <module>
    from pydrive.auth import GoogleAuth
ModuleNotFoundError: No module named 'pydrive'
[2024-05-12T11:43:43.552+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/script.py
[2024-05-12T11:43:43.591+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/script.py took 0.066 seconds
[2024-05-12T11:44:13.771+0000] {processor.py:161} INFO - Started process (PID=177) to work on /opt/airflow/dags/script.py
[2024-05-12T11:44:13.772+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/script.py for tasks to queue
[2024-05-12T11:44:13.774+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:44:13.773+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/script.py
[2024-05-12T11:44:13.789+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:44:13.786+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/script.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/script.py", line 4, in <module>
    from pydrive.auth import GoogleAuth
ModuleNotFoundError: No module named 'pydrive'
[2024-05-12T11:44:13.790+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/script.py
[2024-05-12T11:44:13.818+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/script.py took 0.052 seconds
[2024-05-12T11:44:44.023+0000] {processor.py:161} INFO - Started process (PID=178) to work on /opt/airflow/dags/script.py
[2024-05-12T11:44:44.024+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/script.py for tasks to queue
[2024-05-12T11:44:44.026+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:44:44.026+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/script.py
[2024-05-12T11:44:44.043+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:44:44.040+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/script.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/script.py", line 4, in <module>
    from pydrive.auth import GoogleAuth
ModuleNotFoundError: No module named 'pydrive'
[2024-05-12T11:44:44.044+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/script.py
[2024-05-12T11:44:44.074+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/script.py took 0.057 seconds
[2024-05-12T11:45:12.311+0000] {processor.py:161} INFO - Started process (PID=179) to work on /opt/airflow/dags/script.py
[2024-05-12T11:45:12.312+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/script.py for tasks to queue
[2024-05-12T11:45:12.314+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:45:12.313+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/script.py
[2024-05-12T11:45:12.341+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:45:12.336+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/script.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/script.py", line 4, in <module>
    from pydrive.auth import GoogleAuth
ModuleNotFoundError: No module named 'pydrive'
[2024-05-12T11:45:12.342+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/script.py
[2024-05-12T11:45:12.367+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/script.py took 0.060 seconds
[2024-05-12T11:45:34.569+0000] {processor.py:161} INFO - Started process (PID=180) to work on /opt/airflow/dags/script.py
[2024-05-12T11:45:34.570+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/script.py for tasks to queue
[2024-05-12T11:45:34.573+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:45:34.572+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/script.py
[2024-05-12T11:45:35.774+0000] {processor.py:840} INFO - DAG(s) 'my_dag' retrieved from /opt/airflow/dags/script.py
[2024-05-12T11:45:35.914+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:45:35.913+0000] {override.py:1829} INFO - Created Permission View: can read on DAG:my_dag
[2024-05-12T11:45:35.936+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:45:35.936+0000] {override.py:1829} INFO - Created Permission View: can edit on DAG:my_dag
[2024-05-12T11:45:35.959+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:45:35.959+0000] {override.py:1829} INFO - Created Permission View: can delete on DAG:my_dag
[2024-05-12T11:45:35.960+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:45:35.960+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-12T11:45:35.973+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:45:35.973+0000] {dag.py:3118} INFO - Creating ORM DAG for my_dag
[2024-05-12T11:45:35.982+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:45:35.982+0000] {dag.py:3954} INFO - Setting next_dagrun for my_dag to 2024-04-19 00:00:00+00:00, run_after=2024-04-20 00:00:00+00:00
[2024-05-12T11:45:36.039+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/script.py took 1.476 seconds
[2024-05-12T11:46:15.828+0000] {processor.py:161} INFO - Started process (PID=210) to work on /opt/airflow/dags/script.py
[2024-05-12T11:46:15.829+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/script.py for tasks to queue
[2024-05-12T11:46:15.830+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:46:15.830+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/script.py
[2024-05-12T11:46:16.418+0000] {processor.py:840} INFO - DAG(s) 'my_dag' retrieved from /opt/airflow/dags/script.py
[2024-05-12T11:46:16.443+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:46:16.442+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-12T11:46:16.471+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:46:16.470+0000] {dag.py:3954} INFO - Setting next_dagrun for my_dag to 2024-04-20 00:00:00+00:00, run_after=2024-04-21 00:00:00+00:00
[2024-05-12T11:46:16.517+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/script.py took 0.693 seconds
[2024-05-12T11:46:50.970+0000] {processor.py:161} INFO - Started process (PID=310) to work on /opt/airflow/dags/script.py
[2024-05-12T11:46:50.971+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/script.py for tasks to queue
[2024-05-12T11:46:50.973+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:46:50.972+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/script.py
[2024-05-12T11:46:51.534+0000] {processor.py:840} INFO - DAG(s) 'my_dag' retrieved from /opt/airflow/dags/script.py
[2024-05-12T11:46:51.559+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:46:51.558+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-12T11:46:51.583+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:46:51.583+0000] {dag.py:3954} INFO - Setting next_dagrun for my_dag to 2024-04-22 00:00:00+00:00, run_after=2024-04-23 00:00:00+00:00
[2024-05-12T11:46:51.644+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/script.py took 0.679 seconds
[2024-05-12T11:47:29.902+0000] {processor.py:161} INFO - Started process (PID=424) to work on /opt/airflow/dags/script.py
[2024-05-12T11:47:29.904+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/script.py for tasks to queue
[2024-05-12T11:47:29.905+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:47:29.905+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/script.py
[2024-05-12T11:47:30.503+0000] {processor.py:840} INFO - DAG(s) 'my_dag' retrieved from /opt/airflow/dags/script.py
[2024-05-12T11:47:30.525+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:47:30.524+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-12T11:47:30.549+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:47:30.549+0000] {dag.py:3954} INFO - Setting next_dagrun for my_dag to 2024-04-24 00:00:00+00:00, run_after=2024-04-25 00:00:00+00:00
[2024-05-12T11:47:30.613+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/script.py took 0.716 seconds
[2024-05-12T11:48:09.885+0000] {processor.py:161} INFO - Started process (PID=538) to work on /opt/airflow/dags/script.py
[2024-05-12T11:48:09.885+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/script.py for tasks to queue
[2024-05-12T11:48:09.887+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:48:09.886+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/script.py
[2024-05-12T11:48:10.354+0000] {processor.py:840} INFO - DAG(s) 'my_dag' retrieved from /opt/airflow/dags/script.py
[2024-05-12T11:48:10.374+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:48:10.373+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-12T11:48:10.393+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:48:10.392+0000] {dag.py:3954} INFO - Setting next_dagrun for my_dag to 2024-04-27 00:00:00+00:00, run_after=2024-04-28 00:00:00+00:00
[2024-05-12T11:48:10.434+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/script.py took 0.553 seconds
[2024-05-12T11:48:43.803+0000] {processor.py:161} INFO - Started process (PID=652) to work on /opt/airflow/dags/script.py
[2024-05-12T11:48:43.803+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/script.py for tasks to queue
[2024-05-12T11:48:43.805+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:48:43.804+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/script.py
[2024-05-12T11:48:44.337+0000] {processor.py:840} INFO - DAG(s) 'my_dag' retrieved from /opt/airflow/dags/script.py
[2024-05-12T11:48:44.359+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:48:44.358+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-12T11:48:44.378+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:48:44.377+0000] {dag.py:3954} INFO - Setting next_dagrun for my_dag to 2024-04-29 00:00:00+00:00, run_after=2024-04-30 00:00:00+00:00
[2024-05-12T11:48:44.429+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/script.py took 0.631 seconds
[2024-05-12T11:49:14.804+0000] {processor.py:161} INFO - Started process (PID=738) to work on /opt/airflow/dags/script.py
[2024-05-12T11:49:14.805+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/script.py for tasks to queue
[2024-05-12T11:49:14.807+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:49:14.806+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/script.py
[2024-05-12T11:49:15.336+0000] {processor.py:840} INFO - DAG(s) 'my_dag' retrieved from /opt/airflow/dags/script.py
[2024-05-12T11:49:15.364+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:49:15.364+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-12T11:49:15.393+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:49:15.393+0000] {dag.py:3954} INFO - Setting next_dagrun for my_dag to 2024-05-02 00:00:00+00:00, run_after=2024-05-03 00:00:00+00:00
[2024-05-12T11:49:15.451+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/script.py took 0.652 seconds
[2024-05-12T11:49:58.941+0000] {processor.py:161} INFO - Started process (PID=866) to work on /opt/airflow/dags/script.py
[2024-05-12T11:49:58.942+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/script.py for tasks to queue
[2024-05-12T11:49:58.943+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:49:58.943+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/script.py
[2024-05-12T11:49:59.478+0000] {processor.py:840} INFO - DAG(s) 'my_dag' retrieved from /opt/airflow/dags/script.py
[2024-05-12T11:49:59.506+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:49:59.505+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-12T11:49:59.535+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:49:59.535+0000] {dag.py:3954} INFO - Setting next_dagrun for my_dag to 2024-05-05 00:00:00+00:00, run_after=2024-05-06 00:00:00+00:00
[2024-05-12T11:49:59.604+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/script.py took 0.667 seconds
[2024-05-12T11:50:33.352+0000] {processor.py:161} INFO - Started process (PID=966) to work on /opt/airflow/dags/script.py
[2024-05-12T11:50:33.353+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/script.py for tasks to queue
[2024-05-12T11:50:33.355+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:50:33.355+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/script.py
[2024-05-12T11:50:33.906+0000] {processor.py:840} INFO - DAG(s) 'my_dag' retrieved from /opt/airflow/dags/script.py
[2024-05-12T11:50:33.929+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:50:33.929+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-12T11:50:33.954+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:50:33.953+0000] {dag.py:3954} INFO - Setting next_dagrun for my_dag to 2024-05-07 00:00:00+00:00, run_after=2024-05-08 00:00:00+00:00
[2024-05-12T11:50:34.015+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/script.py took 0.670 seconds
[2024-05-12T11:51:08.551+0000] {processor.py:161} INFO - Started process (PID=1080) to work on /opt/airflow/dags/script.py
[2024-05-12T11:51:08.552+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/script.py for tasks to queue
[2024-05-12T11:51:08.553+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:51:08.553+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/script.py
[2024-05-12T11:51:09.060+0000] {processor.py:840} INFO - DAG(s) 'my_dag' retrieved from /opt/airflow/dags/script.py
[2024-05-12T11:51:09.080+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:51:09.080+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-12T11:51:09.098+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:51:09.098+0000] {dag.py:3954} INFO - Setting next_dagrun for my_dag to 2024-05-10 00:00:00+00:00, run_after=2024-05-11 00:00:00+00:00
[2024-05-12T11:51:09.148+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/script.py took 0.601 seconds
[2024-05-12T11:51:40.772+0000] {processor.py:161} INFO - Started process (PID=1180) to work on /opt/airflow/dags/script.py
[2024-05-12T11:51:40.772+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/script.py for tasks to queue
[2024-05-12T11:51:40.774+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:51:40.774+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/script.py
[2024-05-12T11:51:41.258+0000] {processor.py:840} INFO - DAG(s) 'my_dag' retrieved from /opt/airflow/dags/script.py
[2024-05-12T11:51:41.278+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:51:41.278+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-12T11:51:41.296+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:51:41.296+0000] {dag.py:3954} INFO - Setting next_dagrun for my_dag to 2024-05-12 00:00:00+00:00, run_after=2024-05-13 00:00:00+00:00
[2024-05-12T11:51:41.340+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/script.py took 0.572 seconds
[2024-05-12T11:52:11.868+0000] {processor.py:161} INFO - Started process (PID=1210) to work on /opt/airflow/dags/script.py
[2024-05-12T11:52:11.869+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/script.py for tasks to queue
[2024-05-12T11:52:11.870+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:52:11.870+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/script.py
[2024-05-12T11:52:12.344+0000] {processor.py:840} INFO - DAG(s) 'my_dag' retrieved from /opt/airflow/dags/script.py
[2024-05-12T11:52:12.365+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:52:12.364+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-12T11:52:12.381+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:52:12.381+0000] {dag.py:3954} INFO - Setting next_dagrun for my_dag to 2024-05-12 00:00:00+00:00, run_after=2024-05-13 00:00:00+00:00
[2024-05-12T11:52:12.419+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/script.py took 0.556 seconds
[2024-05-12T11:52:43.073+0000] {processor.py:161} INFO - Started process (PID=1212) to work on /opt/airflow/dags/script.py
[2024-05-12T11:52:43.074+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/script.py for tasks to queue
[2024-05-12T11:52:43.076+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:52:43.076+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/script.py
[2024-05-12T11:52:43.575+0000] {processor.py:840} INFO - DAG(s) 'my_dag' retrieved from /opt/airflow/dags/script.py
[2024-05-12T11:52:43.596+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:52:43.596+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-12T11:52:43.616+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:52:43.616+0000] {dag.py:3954} INFO - Setting next_dagrun for my_dag to 2024-05-12 00:00:00+00:00, run_after=2024-05-13 00:00:00+00:00
[2024-05-12T11:52:43.658+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/script.py took 0.590 seconds
[2024-05-12T11:53:14.253+0000] {processor.py:161} INFO - Started process (PID=1214) to work on /opt/airflow/dags/script.py
[2024-05-12T11:53:14.254+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/script.py for tasks to queue
[2024-05-12T11:53:14.255+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:53:14.255+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/script.py
[2024-05-12T11:53:14.733+0000] {processor.py:840} INFO - DAG(s) 'my_dag' retrieved from /opt/airflow/dags/script.py
[2024-05-12T11:53:14.753+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:53:14.753+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-12T11:53:14.770+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:53:14.770+0000] {dag.py:3954} INFO - Setting next_dagrun for my_dag to 2024-05-12 00:00:00+00:00, run_after=2024-05-13 00:00:00+00:00
[2024-05-12T11:53:14.821+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/script.py took 0.572 seconds
[2024-05-12T11:53:54.011+0000] {processor.py:161} INFO - Started process (PID=64) to work on /opt/airflow/dags/script.py
[2024-05-12T11:53:54.013+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/script.py for tasks to queue
[2024-05-12T11:53:54.017+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:53:54.016+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/script.py
[2024-05-12T11:53:54.570+0000] {processor.py:840} INFO - DAG(s) 'my_dag' retrieved from /opt/airflow/dags/script.py
[2024-05-12T11:53:54.590+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:53:54.590+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-12T11:53:54.840+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:53:54.839+0000] {dag.py:3954} INFO - Setting next_dagrun for my_dag to 2024-05-12 00:00:00+00:00, run_after=2024-05-13 00:00:00+00:00
[2024-05-12T11:53:54.891+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/script.py took 0.892 seconds
[2024-05-12T11:54:38.218+0000] {processor.py:161} INFO - Started process (PID=64) to work on /opt/airflow/dags/script.py
[2024-05-12T11:54:38.220+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/script.py for tasks to queue
[2024-05-12T11:54:38.223+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:54:38.222+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/script.py
[2024-05-12T11:54:38.684+0000] {processor.py:840} INFO - DAG(s) 'my_dag' retrieved from /opt/airflow/dags/script.py
[2024-05-12T11:54:38.696+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:54:38.695+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-12T11:54:38.872+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:54:38.872+0000] {dag.py:3954} INFO - Setting next_dagrun for my_dag to 2024-05-12 00:00:00+00:00, run_after=2024-05-13 00:00:00+00:00
[2024-05-12T11:54:38.929+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/script.py took 0.718 seconds
[2024-05-12T11:55:09.605+0000] {processor.py:161} INFO - Started process (PID=78) to work on /opt/airflow/dags/script.py
[2024-05-12T11:55:09.605+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/script.py for tasks to queue
[2024-05-12T11:55:09.607+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:55:09.606+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/script.py
[2024-05-12T11:55:09.952+0000] {processor.py:840} INFO - DAG(s) 'my_dag' retrieved from /opt/airflow/dags/script.py
[2024-05-12T11:55:10.169+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:55:10.169+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-12T11:55:10.186+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:55:10.186+0000] {dag.py:3954} INFO - Setting next_dagrun for my_dag to 2024-05-12 00:00:00+00:00, run_after=2024-05-13 00:00:00+00:00
[2024-05-12T11:55:10.234+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/script.py took 0.634 seconds
[2024-05-12T11:55:48.181+0000] {processor.py:161} INFO - Started process (PID=64) to work on /opt/airflow/dags/script.py
[2024-05-12T11:55:48.182+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/script.py for tasks to queue
[2024-05-12T11:55:48.183+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:55:48.183+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/script.py
[2024-05-12T11:55:48.646+0000] {processor.py:840} INFO - DAG(s) 'my_dag' retrieved from /opt/airflow/dags/script.py
[2024-05-12T11:55:48.661+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:55:48.660+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-12T11:55:48.940+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:55:48.939+0000] {dag.py:3954} INFO - Setting next_dagrun for my_dag to 2024-05-12 00:00:00+00:00, run_after=2024-05-13 00:00:00+00:00
[2024-05-12T11:55:49.007+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/script.py took 0.830 seconds
[2024-05-12T11:56:19.555+0000] {processor.py:161} INFO - Started process (PID=78) to work on /opt/airflow/dags/script.py
[2024-05-12T11:56:19.557+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/script.py for tasks to queue
[2024-05-12T11:56:19.558+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:56:19.558+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/script.py
[2024-05-12T11:56:19.950+0000] {processor.py:840} INFO - DAG(s) 'my_dag' retrieved from /opt/airflow/dags/script.py
[2024-05-12T11:56:20.178+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:56:20.178+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-12T11:56:20.198+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:56:20.197+0000] {dag.py:3954} INFO - Setting next_dagrun for my_dag to 2024-05-12 00:00:00+00:00, run_after=2024-05-13 00:00:00+00:00
[2024-05-12T11:56:20.250+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/script.py took 0.699 seconds
[2024-05-12T11:56:51.102+0000] {processor.py:161} INFO - Started process (PID=80) to work on /opt/airflow/dags/script.py
[2024-05-12T11:56:51.102+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/script.py for tasks to queue
[2024-05-12T11:56:51.104+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:56:51.103+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/script.py
[2024-05-12T11:56:51.456+0000] {processor.py:840} INFO - DAG(s) 'my_dag' retrieved from /opt/airflow/dags/script.py
[2024-05-12T11:56:51.476+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:56:51.476+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-12T11:56:51.495+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:56:51.494+0000] {dag.py:3954} INFO - Setting next_dagrun for my_dag to 2024-05-12 00:00:00+00:00, run_after=2024-05-13 00:00:00+00:00
[2024-05-12T11:56:51.530+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/script.py took 0.432 seconds
[2024-05-12T11:57:21.212+0000] {processor.py:161} INFO - Started process (PID=64) to work on /opt/airflow/dags/script.py
[2024-05-12T11:57:21.214+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/script.py for tasks to queue
[2024-05-12T11:57:21.216+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:57:21.216+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/script.py
[2024-05-12T11:57:21.699+0000] {processor.py:840} INFO - DAG(s) 'my_dag' retrieved from /opt/airflow/dags/script.py
[2024-05-12T11:57:21.709+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:57:21.708+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-12T11:57:21.883+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:57:21.883+0000] {dag.py:3954} INFO - Setting next_dagrun for my_dag to 2024-05-12 00:00:00+00:00, run_after=2024-05-13 00:00:00+00:00
[2024-05-12T11:57:21.929+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/script.py took 0.722 seconds
[2024-05-12T11:57:52.619+0000] {processor.py:161} INFO - Started process (PID=78) to work on /opt/airflow/dags/script.py
[2024-05-12T11:57:52.620+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/script.py for tasks to queue
[2024-05-12T11:57:52.621+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:57:52.621+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/script.py
[2024-05-12T11:57:52.978+0000] {processor.py:840} INFO - DAG(s) 'my_dag' retrieved from /opt/airflow/dags/script.py
[2024-05-12T11:57:53.198+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:57:53.198+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-12T11:57:53.215+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:57:53.215+0000] {dag.py:3954} INFO - Setting next_dagrun for my_dag to 2024-05-12 00:00:00+00:00, run_after=2024-05-13 00:00:00+00:00
[2024-05-12T11:57:53.260+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/script.py took 0.645 seconds
[2024-05-12T11:59:48.981+0000] {processor.py:161} INFO - Started process (PID=64) to work on /opt/airflow/dags/script.py
[2024-05-12T11:59:48.982+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/script.py for tasks to queue
[2024-05-12T11:59:48.983+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:59:48.983+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/script.py
[2024-05-12T11:59:49.754+0000] {processor.py:840} INFO - DAG(s) 'my_dag' retrieved from /opt/airflow/dags/script.py
[2024-05-12T11:59:49.785+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:59:49.784+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-12T11:59:50.158+0000] {logging_mixin.py:188} INFO - [2024-05-12T11:59:50.158+0000] {dag.py:3954} INFO - Setting next_dagrun for my_dag to 2024-05-12 00:00:00+00:00, run_after=2024-05-13 00:00:00+00:00
[2024-05-12T11:59:50.224+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/script.py took 1.249 seconds
[2024-05-12T12:00:30.696+0000] {processor.py:161} INFO - Started process (PID=64) to work on /opt/airflow/dags/script.py
[2024-05-12T12:00:30.697+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/script.py for tasks to queue
[2024-05-12T12:00:30.698+0000] {logging_mixin.py:188} INFO - [2024-05-12T12:00:30.698+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/script.py
[2024-05-12T12:00:31.069+0000] {processor.py:840} INFO - DAG(s) 'my_dag' retrieved from /opt/airflow/dags/script.py
[2024-05-12T12:00:31.079+0000] {logging_mixin.py:188} INFO - [2024-05-12T12:00:31.078+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-12T12:00:31.234+0000] {logging_mixin.py:188} INFO - [2024-05-12T12:00:31.234+0000] {dag.py:3954} INFO - Setting next_dagrun for my_dag to 2024-05-12 00:00:00+00:00, run_after=2024-05-13 00:00:00+00:00
[2024-05-12T12:00:31.277+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/script.py took 0.587 seconds
[2024-05-12T12:00:48.884+0000] {processor.py:161} INFO - Started process (PID=78) to work on /opt/airflow/dags/script.py
[2024-05-12T12:00:48.884+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/script.py for tasks to queue
[2024-05-12T12:00:48.886+0000] {logging_mixin.py:188} INFO - [2024-05-12T12:00:48.886+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/script.py
[2024-05-12T12:00:48.891+0000] {logging_mixin.py:188} INFO - [2024-05-12T12:00:48.890+0000] {dagbag.py:334} INFO - File /opt/airflow/dags/script.py assumed to contain no DAGs. Skipping.
[2024-05-12T12:00:48.891+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/script.py
[2024-05-12T12:00:48.903+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/script.py took 0.023 seconds
[2024-05-12T12:12:34.164+0000] {processor.py:161} INFO - Started process (PID=86) to work on /opt/airflow/dags/script.py
[2024-05-12T12:12:34.165+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/script.py for tasks to queue
[2024-05-12T12:12:34.167+0000] {logging_mixin.py:188} INFO - [2024-05-12T12:12:34.167+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/script.py
[2024-05-12T12:12:34.198+0000] {logging_mixin.py:188} INFO - [2024-05-12T12:12:34.195+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/script.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/script.py", line 4, in <module>
    from pydrive.auth import GoogleAuth
ModuleNotFoundError: No module named 'pydrive'
[2024-05-12T12:12:34.200+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/script.py
[2024-05-12T12:12:34.235+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/script.py took 0.075 seconds
[2024-05-12T12:12:55.958+0000] {processor.py:161} INFO - Started process (PID=87) to work on /opt/airflow/dags/script.py
[2024-05-12T12:12:55.959+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/script.py for tasks to queue
[2024-05-12T12:12:55.962+0000] {logging_mixin.py:188} INFO - [2024-05-12T12:12:55.961+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/script.py
[2024-05-12T12:12:56.004+0000] {logging_mixin.py:188} INFO - [2024-05-12T12:12:56.000+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/script.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/script.py", line 4, in <module>
    from pydrive.auth import GoogleAuth
ModuleNotFoundError: No module named 'pydrive'
[2024-05-12T12:12:56.006+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/script.py
[2024-05-12T12:12:56.040+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/script.py took 0.089 seconds
[2024-05-12T12:12:59.440+0000] {processor.py:161} INFO - Started process (PID=88) to work on /opt/airflow/dags/script.py
[2024-05-12T12:12:59.441+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/script.py for tasks to queue
[2024-05-12T12:12:59.443+0000] {logging_mixin.py:188} INFO - [2024-05-12T12:12:59.442+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/script.py
[2024-05-12T12:12:59.898+0000] {processor.py:840} INFO - DAG(s) 'my_dag' retrieved from /opt/airflow/dags/script.py
[2024-05-12T12:13:00.149+0000] {logging_mixin.py:188} INFO - [2024-05-12T12:13:00.149+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-12T12:13:00.169+0000] {logging_mixin.py:188} INFO - [2024-05-12T12:13:00.168+0000] {dag.py:3954} INFO - Setting next_dagrun for my_dag to 2024-05-12 00:00:00+00:00, run_after=2024-05-13 00:00:00+00:00
[2024-05-12T12:13:00.221+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/script.py took 0.785 seconds
[2024-05-12T12:13:30.692+0000] {processor.py:161} INFO - Started process (PID=90) to work on /opt/airflow/dags/script.py
[2024-05-12T12:13:30.693+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/script.py for tasks to queue
[2024-05-12T12:13:30.694+0000] {logging_mixin.py:188} INFO - [2024-05-12T12:13:30.694+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/script.py
[2024-05-12T12:13:31.046+0000] {processor.py:840} INFO - DAG(s) 'my_dag' retrieved from /opt/airflow/dags/script.py
[2024-05-12T12:13:31.070+0000] {logging_mixin.py:188} INFO - [2024-05-12T12:13:31.069+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-12T12:13:31.092+0000] {logging_mixin.py:188} INFO - [2024-05-12T12:13:31.092+0000] {dag.py:3954} INFO - Setting next_dagrun for my_dag to 2024-05-12 00:00:00+00:00, run_after=2024-05-13 00:00:00+00:00
[2024-05-12T12:13:31.149+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/script.py took 0.461 seconds
[2024-05-12T12:14:01.765+0000] {processor.py:161} INFO - Started process (PID=92) to work on /opt/airflow/dags/script.py
[2024-05-12T12:14:01.766+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/script.py for tasks to queue
[2024-05-12T12:14:01.767+0000] {logging_mixin.py:188} INFO - [2024-05-12T12:14:01.767+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/script.py
[2024-05-12T12:14:02.256+0000] {processor.py:840} INFO - DAG(s) 'my_dag' retrieved from /opt/airflow/dags/script.py
[2024-05-12T12:14:02.278+0000] {logging_mixin.py:188} INFO - [2024-05-12T12:14:02.277+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-12T12:14:02.296+0000] {logging_mixin.py:188} INFO - [2024-05-12T12:14:02.295+0000] {dag.py:3954} INFO - Setting next_dagrun for my_dag to 2024-05-12 00:00:00+00:00, run_after=2024-05-13 00:00:00+00:00
[2024-05-12T12:14:02.335+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/script.py took 0.574 seconds
[2024-05-12T12:14:32.924+0000] {processor.py:161} INFO - Started process (PID=94) to work on /opt/airflow/dags/script.py
[2024-05-12T12:14:32.925+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/script.py for tasks to queue
[2024-05-12T12:14:32.926+0000] {logging_mixin.py:188} INFO - [2024-05-12T12:14:32.926+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/script.py
[2024-05-12T12:14:33.428+0000] {processor.py:840} INFO - DAG(s) 'my_dag' retrieved from /opt/airflow/dags/script.py
[2024-05-12T12:14:33.453+0000] {logging_mixin.py:188} INFO - [2024-05-12T12:14:33.453+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-12T12:14:33.479+0000] {logging_mixin.py:188} INFO - [2024-05-12T12:14:33.478+0000] {dag.py:3954} INFO - Setting next_dagrun for my_dag to 2024-05-12 00:00:00+00:00, run_after=2024-05-13 00:00:00+00:00
[2024-05-12T12:14:33.540+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/script.py took 0.621 seconds
[2024-05-12T12:15:04.201+0000] {processor.py:161} INFO - Started process (PID=96) to work on /opt/airflow/dags/script.py
[2024-05-12T12:15:04.202+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/script.py for tasks to queue
[2024-05-12T12:15:04.204+0000] {logging_mixin.py:188} INFO - [2024-05-12T12:15:04.203+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/script.py
[2024-05-12T12:15:04.677+0000] {processor.py:840} INFO - DAG(s) 'my_dag' retrieved from /opt/airflow/dags/script.py
[2024-05-12T12:15:04.697+0000] {logging_mixin.py:188} INFO - [2024-05-12T12:15:04.696+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-12T12:15:04.719+0000] {logging_mixin.py:188} INFO - [2024-05-12T12:15:04.718+0000] {dag.py:3954} INFO - Setting next_dagrun for my_dag to 2024-05-12 00:00:00+00:00, run_after=2024-05-13 00:00:00+00:00
[2024-05-12T12:15:04.768+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/script.py took 0.571 seconds
[2024-05-12T12:15:35.359+0000] {processor.py:161} INFO - Started process (PID=98) to work on /opt/airflow/dags/script.py
[2024-05-12T12:15:35.360+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/script.py for tasks to queue
[2024-05-12T12:15:35.362+0000] {logging_mixin.py:188} INFO - [2024-05-12T12:15:35.361+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/script.py
[2024-05-12T12:15:35.910+0000] {processor.py:840} INFO - DAG(s) 'my_dag' retrieved from /opt/airflow/dags/script.py
[2024-05-12T12:15:35.932+0000] {logging_mixin.py:188} INFO - [2024-05-12T12:15:35.931+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-12T12:15:35.952+0000] {logging_mixin.py:188} INFO - [2024-05-12T12:15:35.952+0000] {dag.py:3954} INFO - Setting next_dagrun for my_dag to 2024-05-12 00:00:00+00:00, run_after=2024-05-13 00:00:00+00:00
[2024-05-12T12:15:35.991+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/script.py took 0.636 seconds
[2024-05-12T12:16:06.603+0000] {processor.py:161} INFO - Started process (PID=100) to work on /opt/airflow/dags/script.py
[2024-05-12T12:16:06.608+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/script.py for tasks to queue
[2024-05-12T12:16:06.610+0000] {logging_mixin.py:188} INFO - [2024-05-12T12:16:06.609+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/script.py
[2024-05-12T12:16:06.952+0000] {processor.py:840} INFO - DAG(s) 'my_dag' retrieved from /opt/airflow/dags/script.py
[2024-05-12T12:16:07.107+0000] {logging_mixin.py:188} INFO - [2024-05-12T12:16:07.107+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-12T12:16:07.125+0000] {logging_mixin.py:188} INFO - [2024-05-12T12:16:07.125+0000] {dag.py:3954} INFO - Setting next_dagrun for my_dag to 2024-05-12 00:00:00+00:00, run_after=2024-05-13 00:00:00+00:00
[2024-05-12T12:16:07.174+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/script.py took 0.575 seconds
[2024-05-12T12:16:37.600+0000] {processor.py:161} INFO - Started process (PID=102) to work on /opt/airflow/dags/script.py
[2024-05-12T12:16:37.601+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/script.py for tasks to queue
[2024-05-12T12:16:37.602+0000] {logging_mixin.py:188} INFO - [2024-05-12T12:16:37.602+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/script.py
[2024-05-12T12:16:37.953+0000] {processor.py:840} INFO - DAG(s) 'my_dag' retrieved from /opt/airflow/dags/script.py
[2024-05-12T12:16:37.974+0000] {logging_mixin.py:188} INFO - [2024-05-12T12:16:37.973+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-12T12:16:37.993+0000] {logging_mixin.py:188} INFO - [2024-05-12T12:16:37.993+0000] {dag.py:3954} INFO - Setting next_dagrun for my_dag to 2024-05-12 00:00:00+00:00, run_after=2024-05-13 00:00:00+00:00
[2024-05-12T12:16:38.031+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/script.py took 0.436 seconds
[2024-05-12T12:17:08.262+0000] {processor.py:161} INFO - Started process (PID=104) to work on /opt/airflow/dags/script.py
[2024-05-12T12:17:08.262+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/script.py for tasks to queue
[2024-05-12T12:17:08.264+0000] {logging_mixin.py:188} INFO - [2024-05-12T12:17:08.263+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/script.py
[2024-05-12T12:17:08.733+0000] {processor.py:840} INFO - DAG(s) 'my_dag' retrieved from /opt/airflow/dags/script.py
[2024-05-12T12:17:08.753+0000] {logging_mixin.py:188} INFO - [2024-05-12T12:17:08.752+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-12T12:17:08.771+0000] {logging_mixin.py:188} INFO - [2024-05-12T12:17:08.771+0000] {dag.py:3954} INFO - Setting next_dagrun for my_dag to 2024-05-12 00:00:00+00:00, run_after=2024-05-13 00:00:00+00:00
[2024-05-12T12:17:08.821+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/script.py took 0.563 seconds
[2024-05-12T12:17:39.030+0000] {processor.py:161} INFO - Started process (PID=106) to work on /opt/airflow/dags/script.py
[2024-05-12T12:17:39.031+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/script.py for tasks to queue
[2024-05-12T12:17:39.032+0000] {logging_mixin.py:188} INFO - [2024-05-12T12:17:39.032+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/script.py
[2024-05-12T12:17:39.540+0000] {processor.py:840} INFO - DAG(s) 'my_dag' retrieved from /opt/airflow/dags/script.py
[2024-05-12T12:17:39.559+0000] {logging_mixin.py:188} INFO - [2024-05-12T12:17:39.559+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-12T12:17:39.579+0000] {logging_mixin.py:188} INFO - [2024-05-12T12:17:39.579+0000] {dag.py:3954} INFO - Setting next_dagrun for my_dag to 2024-05-12 00:00:00+00:00, run_after=2024-05-13 00:00:00+00:00
[2024-05-12T12:17:39.631+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/script.py took 0.606 seconds
[2024-05-12T12:18:09.900+0000] {processor.py:161} INFO - Started process (PID=108) to work on /opt/airflow/dags/script.py
[2024-05-12T12:18:09.901+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/script.py for tasks to queue
[2024-05-12T12:18:09.902+0000] {logging_mixin.py:188} INFO - [2024-05-12T12:18:09.902+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/script.py
[2024-05-12T12:18:10.402+0000] {processor.py:840} INFO - DAG(s) 'my_dag' retrieved from /opt/airflow/dags/script.py
[2024-05-12T12:18:10.424+0000] {logging_mixin.py:188} INFO - [2024-05-12T12:18:10.424+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-12T12:18:10.445+0000] {logging_mixin.py:188} INFO - [2024-05-12T12:18:10.445+0000] {dag.py:3954} INFO - Setting next_dagrun for my_dag to 2024-05-12 00:00:00+00:00, run_after=2024-05-13 00:00:00+00:00
[2024-05-12T12:18:10.491+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/script.py took 0.596 seconds
[2024-05-12T12:18:40.849+0000] {processor.py:161} INFO - Started process (PID=110) to work on /opt/airflow/dags/script.py
[2024-05-12T12:18:40.850+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/script.py for tasks to queue
[2024-05-12T12:18:40.853+0000] {logging_mixin.py:188} INFO - [2024-05-12T12:18:40.853+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/script.py
[2024-05-12T12:18:41.869+0000] {processor.py:840} INFO - DAG(s) 'my_dag' retrieved from /opt/airflow/dags/script.py
[2024-05-12T12:18:41.931+0000] {logging_mixin.py:188} INFO - [2024-05-12T12:18:41.930+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-12T12:18:41.981+0000] {logging_mixin.py:188} INFO - [2024-05-12T12:18:41.980+0000] {dag.py:3954} INFO - Setting next_dagrun for my_dag to 2024-05-12 00:00:00+00:00, run_after=2024-05-13 00:00:00+00:00
[2024-05-12T12:18:42.089+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/script.py took 1.247 seconds
[2024-05-12T12:19:12.366+0000] {processor.py:161} INFO - Started process (PID=112) to work on /opt/airflow/dags/script.py
[2024-05-12T12:19:12.367+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/script.py for tasks to queue
[2024-05-12T12:19:12.368+0000] {logging_mixin.py:188} INFO - [2024-05-12T12:19:12.368+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/script.py
[2024-05-12T12:19:12.729+0000] {processor.py:840} INFO - DAG(s) 'my_dag' retrieved from /opt/airflow/dags/script.py
[2024-05-12T12:19:12.882+0000] {logging_mixin.py:188} INFO - [2024-05-12T12:19:12.881+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-12T12:19:12.899+0000] {logging_mixin.py:188} INFO - [2024-05-12T12:19:12.899+0000] {dag.py:3954} INFO - Setting next_dagrun for my_dag to 2024-05-12 00:00:00+00:00, run_after=2024-05-13 00:00:00+00:00
[2024-05-12T12:19:12.946+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/script.py took 0.584 seconds
[2024-05-12T12:19:43.138+0000] {processor.py:161} INFO - Started process (PID=114) to work on /opt/airflow/dags/script.py
[2024-05-12T12:19:43.139+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/script.py for tasks to queue
[2024-05-12T12:19:43.142+0000] {logging_mixin.py:188} INFO - [2024-05-12T12:19:43.141+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/script.py
[2024-05-12T12:19:43.987+0000] {processor.py:840} INFO - DAG(s) 'my_dag' retrieved from /opt/airflow/dags/script.py
[2024-05-12T12:19:44.020+0000] {logging_mixin.py:188} INFO - [2024-05-12T12:19:44.019+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-12T12:19:44.048+0000] {logging_mixin.py:188} INFO - [2024-05-12T12:19:44.048+0000] {dag.py:3954} INFO - Setting next_dagrun for my_dag to 2024-05-12 00:00:00+00:00, run_after=2024-05-13 00:00:00+00:00
[2024-05-12T12:19:44.113+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/script.py took 0.981 seconds
[2024-05-12T12:20:14.302+0000] {processor.py:161} INFO - Started process (PID=116) to work on /opt/airflow/dags/script.py
[2024-05-12T12:20:14.303+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/script.py for tasks to queue
[2024-05-12T12:20:14.304+0000] {logging_mixin.py:188} INFO - [2024-05-12T12:20:14.304+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/script.py
[2024-05-12T12:20:14.804+0000] {processor.py:840} INFO - DAG(s) 'my_dag' retrieved from /opt/airflow/dags/script.py
[2024-05-12T12:20:14.824+0000] {logging_mixin.py:188} INFO - [2024-05-12T12:20:14.823+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-12T12:20:14.842+0000] {logging_mixin.py:188} INFO - [2024-05-12T12:20:14.842+0000] {dag.py:3954} INFO - Setting next_dagrun for my_dag to 2024-05-12 00:00:00+00:00, run_after=2024-05-13 00:00:00+00:00
[2024-05-12T12:20:14.879+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/script.py took 0.580 seconds
[2024-05-12T12:20:45.061+0000] {processor.py:161} INFO - Started process (PID=118) to work on /opt/airflow/dags/script.py
[2024-05-12T12:20:45.061+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/script.py for tasks to queue
[2024-05-12T12:20:45.063+0000] {logging_mixin.py:188} INFO - [2024-05-12T12:20:45.062+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/script.py
[2024-05-12T12:20:45.531+0000] {processor.py:840} INFO - DAG(s) 'my_dag' retrieved from /opt/airflow/dags/script.py
[2024-05-12T12:20:45.550+0000] {logging_mixin.py:188} INFO - [2024-05-12T12:20:45.549+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-12T12:20:45.569+0000] {logging_mixin.py:188} INFO - [2024-05-12T12:20:45.569+0000] {dag.py:3954} INFO - Setting next_dagrun for my_dag to 2024-05-12 00:00:00+00:00, run_after=2024-05-13 00:00:00+00:00
[2024-05-12T12:20:45.607+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/script.py took 0.551 seconds
[2024-05-12T12:21:15.839+0000] {processor.py:161} INFO - Started process (PID=120) to work on /opt/airflow/dags/script.py
[2024-05-12T12:21:15.840+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/script.py for tasks to queue
[2024-05-12T12:21:15.842+0000] {logging_mixin.py:188} INFO - [2024-05-12T12:21:15.841+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/script.py
[2024-05-12T12:21:16.336+0000] {processor.py:840} INFO - DAG(s) 'my_dag' retrieved from /opt/airflow/dags/script.py
[2024-05-12T12:21:16.358+0000] {logging_mixin.py:188} INFO - [2024-05-12T12:21:16.358+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-12T12:21:16.378+0000] {logging_mixin.py:188} INFO - [2024-05-12T12:21:16.377+0000] {dag.py:3954} INFO - Setting next_dagrun for my_dag to 2024-05-12 00:00:00+00:00, run_after=2024-05-13 00:00:00+00:00
[2024-05-12T12:21:16.416+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/script.py took 0.581 seconds
[2024-05-12T12:21:46.613+0000] {processor.py:161} INFO - Started process (PID=122) to work on /opt/airflow/dags/script.py
[2024-05-12T12:21:46.614+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/script.py for tasks to queue
[2024-05-12T12:21:46.616+0000] {logging_mixin.py:188} INFO - [2024-05-12T12:21:46.615+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/script.py
[2024-05-12T12:21:47.124+0000] {processor.py:840} INFO - DAG(s) 'my_dag' retrieved from /opt/airflow/dags/script.py
[2024-05-12T12:21:47.143+0000] {logging_mixin.py:188} INFO - [2024-05-12T12:21:47.142+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-12T12:21:47.161+0000] {logging_mixin.py:188} INFO - [2024-05-12T12:21:47.161+0000] {dag.py:3954} INFO - Setting next_dagrun for my_dag to 2024-05-12 00:00:00+00:00, run_after=2024-05-13 00:00:00+00:00
[2024-05-12T12:21:47.209+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/script.py took 0.601 seconds
[2024-05-12T12:22:17.470+0000] {processor.py:161} INFO - Started process (PID=124) to work on /opt/airflow/dags/script.py
[2024-05-12T12:22:17.471+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/script.py for tasks to queue
[2024-05-12T12:22:17.472+0000] {logging_mixin.py:188} INFO - [2024-05-12T12:22:17.472+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/script.py
[2024-05-12T12:22:17.939+0000] {processor.py:840} INFO - DAG(s) 'my_dag' retrieved from /opt/airflow/dags/script.py
[2024-05-12T12:22:17.960+0000] {logging_mixin.py:188} INFO - [2024-05-12T12:22:17.960+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-12T12:22:17.978+0000] {logging_mixin.py:188} INFO - [2024-05-12T12:22:17.978+0000] {dag.py:3954} INFO - Setting next_dagrun for my_dag to 2024-05-12 00:00:00+00:00, run_after=2024-05-13 00:00:00+00:00
[2024-05-12T12:22:18.026+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/script.py took 0.560 seconds
[2024-05-12T12:22:48.236+0000] {processor.py:161} INFO - Started process (PID=126) to work on /opt/airflow/dags/script.py
[2024-05-12T12:22:48.237+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/script.py for tasks to queue
[2024-05-12T12:22:48.238+0000] {logging_mixin.py:188} INFO - [2024-05-12T12:22:48.238+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/script.py
[2024-05-12T12:22:48.583+0000] {processor.py:840} INFO - DAG(s) 'my_dag' retrieved from /opt/airflow/dags/script.py
[2024-05-12T12:22:48.748+0000] {logging_mixin.py:188} INFO - [2024-05-12T12:22:48.748+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-12T12:22:48.770+0000] {logging_mixin.py:188} INFO - [2024-05-12T12:22:48.769+0000] {dag.py:3954} INFO - Setting next_dagrun for my_dag to 2024-05-12 00:00:00+00:00, run_after=2024-05-13 00:00:00+00:00
[2024-05-12T12:22:48.817+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/script.py took 0.584 seconds
[2024-05-12T12:23:18.994+0000] {processor.py:161} INFO - Started process (PID=128) to work on /opt/airflow/dags/script.py
[2024-05-12T12:23:18.995+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/script.py for tasks to queue
[2024-05-12T12:23:18.996+0000] {logging_mixin.py:188} INFO - [2024-05-12T12:23:18.996+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/script.py
[2024-05-12T12:23:19.466+0000] {processor.py:840} INFO - DAG(s) 'my_dag' retrieved from /opt/airflow/dags/script.py
[2024-05-12T12:23:19.485+0000] {logging_mixin.py:188} INFO - [2024-05-12T12:23:19.485+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-12T12:23:19.503+0000] {logging_mixin.py:188} INFO - [2024-05-12T12:23:19.503+0000] {dag.py:3954} INFO - Setting next_dagrun for my_dag to 2024-05-12 00:00:00+00:00, run_after=2024-05-13 00:00:00+00:00
[2024-05-12T12:23:19.551+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/script.py took 0.561 seconds
[2024-05-12T12:23:49.754+0000] {processor.py:161} INFO - Started process (PID=130) to work on /opt/airflow/dags/script.py
[2024-05-12T12:23:49.755+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/script.py for tasks to queue
[2024-05-12T12:23:49.756+0000] {logging_mixin.py:188} INFO - [2024-05-12T12:23:49.756+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/script.py
[2024-05-12T12:23:50.210+0000] {processor.py:840} INFO - DAG(s) 'my_dag' retrieved from /opt/airflow/dags/script.py
[2024-05-12T12:23:50.229+0000] {logging_mixin.py:188} INFO - [2024-05-12T12:23:50.228+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-12T12:23:50.246+0000] {logging_mixin.py:188} INFO - [2024-05-12T12:23:50.246+0000] {dag.py:3954} INFO - Setting next_dagrun for my_dag to 2024-05-12 00:00:00+00:00, run_after=2024-05-13 00:00:00+00:00
[2024-05-12T12:23:50.296+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/script.py took 0.546 seconds
[2024-05-12T12:24:20.516+0000] {processor.py:161} INFO - Started process (PID=132) to work on /opt/airflow/dags/script.py
[2024-05-12T12:24:20.517+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/script.py for tasks to queue
[2024-05-12T12:24:20.518+0000] {logging_mixin.py:188} INFO - [2024-05-12T12:24:20.518+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/script.py
[2024-05-12T12:24:21.003+0000] {processor.py:840} INFO - DAG(s) 'my_dag' retrieved from /opt/airflow/dags/script.py
[2024-05-12T12:24:21.023+0000] {logging_mixin.py:188} INFO - [2024-05-12T12:24:21.023+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-12T12:24:21.042+0000] {logging_mixin.py:188} INFO - [2024-05-12T12:24:21.042+0000] {dag.py:3954} INFO - Setting next_dagrun for my_dag to 2024-05-12 00:00:00+00:00, run_after=2024-05-13 00:00:00+00:00
[2024-05-12T12:24:21.092+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/script.py took 0.579 seconds
[2024-05-12T12:24:51.292+0000] {processor.py:161} INFO - Started process (PID=134) to work on /opt/airflow/dags/script.py
[2024-05-12T12:24:51.293+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/script.py for tasks to queue
[2024-05-12T12:24:51.296+0000] {logging_mixin.py:188} INFO - [2024-05-12T12:24:51.295+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/script.py
[2024-05-12T12:24:51.922+0000] {processor.py:840} INFO - DAG(s) 'my_dag' retrieved from /opt/airflow/dags/script.py
[2024-05-12T12:24:51.956+0000] {logging_mixin.py:188} INFO - [2024-05-12T12:24:51.955+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-12T12:24:51.982+0000] {logging_mixin.py:188} INFO - [2024-05-12T12:24:51.981+0000] {dag.py:3954} INFO - Setting next_dagrun for my_dag to 2024-05-12 00:00:00+00:00, run_after=2024-05-13 00:00:00+00:00
[2024-05-12T12:24:52.033+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/script.py took 0.748 seconds
[2024-05-12T12:25:22.249+0000] {processor.py:161} INFO - Started process (PID=136) to work on /opt/airflow/dags/script.py
[2024-05-12T12:25:22.250+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/script.py for tasks to queue
[2024-05-12T12:25:22.251+0000] {logging_mixin.py:188} INFO - [2024-05-12T12:25:22.251+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/script.py
[2024-05-12T12:25:22.739+0000] {processor.py:840} INFO - DAG(s) 'my_dag' retrieved from /opt/airflow/dags/script.py
[2024-05-12T12:25:22.758+0000] {logging_mixin.py:188} INFO - [2024-05-12T12:25:22.757+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-12T12:25:22.777+0000] {logging_mixin.py:188} INFO - [2024-05-12T12:25:22.777+0000] {dag.py:3954} INFO - Setting next_dagrun for my_dag to 2024-05-12 00:00:00+00:00, run_after=2024-05-13 00:00:00+00:00
[2024-05-12T12:25:22.825+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/script.py took 0.579 seconds
[2024-05-12T12:25:53.011+0000] {processor.py:161} INFO - Started process (PID=138) to work on /opt/airflow/dags/script.py
[2024-05-12T12:25:53.012+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/script.py for tasks to queue
[2024-05-12T12:25:53.013+0000] {logging_mixin.py:188} INFO - [2024-05-12T12:25:53.013+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/script.py
[2024-05-12T12:25:53.503+0000] {processor.py:840} INFO - DAG(s) 'my_dag' retrieved from /opt/airflow/dags/script.py
[2024-05-12T12:25:53.524+0000] {logging_mixin.py:188} INFO - [2024-05-12T12:25:53.523+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-12T12:25:53.541+0000] {logging_mixin.py:188} INFO - [2024-05-12T12:25:53.541+0000] {dag.py:3954} INFO - Setting next_dagrun for my_dag to 2024-05-12 00:00:00+00:00, run_after=2024-05-13 00:00:00+00:00
[2024-05-12T12:25:53.590+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/script.py took 0.583 seconds
[2024-05-12T12:26:23.785+0000] {processor.py:161} INFO - Started process (PID=140) to work on /opt/airflow/dags/script.py
[2024-05-12T12:26:23.787+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/script.py for tasks to queue
[2024-05-12T12:26:23.789+0000] {logging_mixin.py:188} INFO - [2024-05-12T12:26:23.789+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/script.py
[2024-05-12T12:26:24.592+0000] {processor.py:840} INFO - DAG(s) 'my_dag' retrieved from /opt/airflow/dags/script.py
[2024-05-12T12:26:24.614+0000] {logging_mixin.py:188} INFO - [2024-05-12T12:26:24.614+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-12T12:26:24.632+0000] {logging_mixin.py:188} INFO - [2024-05-12T12:26:24.632+0000] {dag.py:3954} INFO - Setting next_dagrun for my_dag to 2024-05-12 00:00:00+00:00, run_after=2024-05-13 00:00:00+00:00
[2024-05-12T12:26:24.672+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/script.py took 0.894 seconds
[2024-05-12T12:26:54.941+0000] {processor.py:161} INFO - Started process (PID=142) to work on /opt/airflow/dags/script.py
[2024-05-12T12:26:54.942+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/script.py for tasks to queue
[2024-05-12T12:26:54.943+0000] {logging_mixin.py:188} INFO - [2024-05-12T12:26:54.943+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/script.py
[2024-05-12T12:26:55.431+0000] {processor.py:840} INFO - DAG(s) 'my_dag' retrieved from /opt/airflow/dags/script.py
[2024-05-12T12:26:55.453+0000] {logging_mixin.py:188} INFO - [2024-05-12T12:26:55.452+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-12T12:26:55.471+0000] {logging_mixin.py:188} INFO - [2024-05-12T12:26:55.470+0000] {dag.py:3954} INFO - Setting next_dagrun for my_dag to 2024-05-12 00:00:00+00:00, run_after=2024-05-13 00:00:00+00:00
[2024-05-12T12:26:55.509+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/script.py took 0.572 seconds
[2024-05-12T12:27:25.700+0000] {processor.py:161} INFO - Started process (PID=144) to work on /opt/airflow/dags/script.py
[2024-05-12T12:27:25.700+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/script.py for tasks to queue
[2024-05-12T12:27:25.702+0000] {logging_mixin.py:188} INFO - [2024-05-12T12:27:25.701+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/script.py
[2024-05-12T12:27:26.212+0000] {processor.py:840} INFO - DAG(s) 'my_dag' retrieved from /opt/airflow/dags/script.py
[2024-05-12T12:27:26.234+0000] {logging_mixin.py:188} INFO - [2024-05-12T12:27:26.234+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-12T12:27:26.252+0000] {logging_mixin.py:188} INFO - [2024-05-12T12:27:26.252+0000] {dag.py:3954} INFO - Setting next_dagrun for my_dag to 2024-05-12 00:00:00+00:00, run_after=2024-05-13 00:00:00+00:00
[2024-05-12T12:27:26.309+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/script.py took 0.614 seconds
[2024-05-12T12:27:56.567+0000] {processor.py:161} INFO - Started process (PID=146) to work on /opt/airflow/dags/script.py
[2024-05-12T12:27:56.568+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/script.py for tasks to queue
[2024-05-12T12:27:56.570+0000] {logging_mixin.py:188} INFO - [2024-05-12T12:27:56.569+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/script.py
[2024-05-12T12:27:57.059+0000] {processor.py:840} INFO - DAG(s) 'my_dag' retrieved from /opt/airflow/dags/script.py
[2024-05-12T12:27:57.080+0000] {logging_mixin.py:188} INFO - [2024-05-12T12:27:57.080+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-12T12:27:57.101+0000] {logging_mixin.py:188} INFO - [2024-05-12T12:27:57.101+0000] {dag.py:3954} INFO - Setting next_dagrun for my_dag to 2024-05-12 00:00:00+00:00, run_after=2024-05-13 00:00:00+00:00
[2024-05-12T12:27:57.151+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/script.py took 0.587 seconds
[2024-05-12T12:28:27.334+0000] {processor.py:161} INFO - Started process (PID=148) to work on /opt/airflow/dags/script.py
[2024-05-12T12:28:27.334+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/script.py for tasks to queue
[2024-05-12T12:28:27.336+0000] {logging_mixin.py:188} INFO - [2024-05-12T12:28:27.336+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/script.py
[2024-05-12T12:28:27.807+0000] {processor.py:840} INFO - DAG(s) 'my_dag' retrieved from /opt/airflow/dags/script.py
[2024-05-12T12:28:27.826+0000] {logging_mixin.py:188} INFO - [2024-05-12T12:28:27.826+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-12T12:28:27.845+0000] {logging_mixin.py:188} INFO - [2024-05-12T12:28:27.845+0000] {dag.py:3954} INFO - Setting next_dagrun for my_dag to 2024-05-12 00:00:00+00:00, run_after=2024-05-13 00:00:00+00:00
[2024-05-12T12:28:27.893+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/script.py took 0.563 seconds
[2024-05-12T12:36:36.507+0000] {processor.py:161} INFO - Started process (PID=150) to work on /opt/airflow/dags/script.py
[2024-05-12T12:36:36.508+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/script.py for tasks to queue
[2024-05-12T12:36:36.509+0000] {logging_mixin.py:188} INFO - [2024-05-12T12:36:36.509+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/script.py
[2024-05-12T12:36:37.318+0000] {processor.py:840} INFO - DAG(s) 'my_dag' retrieved from /opt/airflow/dags/script.py
[2024-05-12T12:36:37.348+0000] {logging_mixin.py:188} INFO - [2024-05-12T12:36:37.347+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-12T12:36:37.374+0000] {logging_mixin.py:188} INFO - [2024-05-12T12:36:37.374+0000] {dag.py:3954} INFO - Setting next_dagrun for my_dag to 2024-05-12 00:00:00+00:00, run_after=2024-05-13 00:00:00+00:00
[2024-05-12T12:36:37.433+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/script.py took 0.930 seconds
[2024-05-12T12:37:33.024+0000] {processor.py:161} INFO - Started process (PID=152) to work on /opt/airflow/dags/script.py
[2024-05-12T12:37:33.025+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/script.py for tasks to queue
[2024-05-12T12:37:33.026+0000] {logging_mixin.py:188} INFO - [2024-05-12T12:37:33.026+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/script.py
[2024-05-12T12:37:33.502+0000] {processor.py:840} INFO - DAG(s) 'my_dag' retrieved from /opt/airflow/dags/script.py
[2024-05-12T12:37:33.522+0000] {logging_mixin.py:188} INFO - [2024-05-12T12:37:33.522+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-12T12:37:33.540+0000] {logging_mixin.py:188} INFO - [2024-05-12T12:37:33.540+0000] {dag.py:3954} INFO - Setting next_dagrun for my_dag to 2024-05-12 00:00:00+00:00, run_after=2024-05-13 00:00:00+00:00
[2024-05-12T12:37:33.578+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/script.py took 0.557 seconds
[2024-05-12T12:38:04.495+0000] {processor.py:161} INFO - Started process (PID=154) to work on /opt/airflow/dags/script.py
[2024-05-12T12:38:04.496+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/script.py for tasks to queue
[2024-05-12T12:38:04.497+0000] {logging_mixin.py:188} INFO - [2024-05-12T12:38:04.497+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/script.py
[2024-05-12T12:38:04.987+0000] {processor.py:840} INFO - DAG(s) 'my_dag' retrieved from /opt/airflow/dags/script.py
[2024-05-12T12:38:05.007+0000] {logging_mixin.py:188} INFO - [2024-05-12T12:38:05.007+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-12T12:38:05.025+0000] {logging_mixin.py:188} INFO - [2024-05-12T12:38:05.025+0000] {dag.py:3954} INFO - Setting next_dagrun for my_dag to 2024-05-12 00:00:00+00:00, run_after=2024-05-13 00:00:00+00:00
[2024-05-12T12:38:05.073+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/script.py took 0.582 seconds
[2024-05-12T12:38:35.840+0000] {processor.py:161} INFO - Started process (PID=156) to work on /opt/airflow/dags/script.py
[2024-05-12T12:38:35.841+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/script.py for tasks to queue
[2024-05-12T12:38:35.842+0000] {logging_mixin.py:188} INFO - [2024-05-12T12:38:35.842+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/script.py
[2024-05-12T12:38:36.318+0000] {processor.py:840} INFO - DAG(s) 'my_dag' retrieved from /opt/airflow/dags/script.py
[2024-05-12T12:38:36.337+0000] {logging_mixin.py:188} INFO - [2024-05-12T12:38:36.337+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-12T12:38:36.356+0000] {logging_mixin.py:188} INFO - [2024-05-12T12:38:36.355+0000] {dag.py:3954} INFO - Setting next_dagrun for my_dag to 2024-05-12 00:00:00+00:00, run_after=2024-05-13 00:00:00+00:00
[2024-05-12T12:38:36.414+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/script.py took 0.577 seconds
[2024-05-12T12:39:06.984+0000] {processor.py:161} INFO - Started process (PID=158) to work on /opt/airflow/dags/script.py
[2024-05-12T12:39:06.984+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/script.py for tasks to queue
[2024-05-12T12:39:06.986+0000] {logging_mixin.py:188} INFO - [2024-05-12T12:39:06.985+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/script.py
[2024-05-12T12:39:07.431+0000] {processor.py:840} INFO - DAG(s) 'my_dag' retrieved from /opt/airflow/dags/script.py
[2024-05-12T12:39:07.450+0000] {logging_mixin.py:188} INFO - [2024-05-12T12:39:07.449+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-12T12:39:07.468+0000] {logging_mixin.py:188} INFO - [2024-05-12T12:39:07.468+0000] {dag.py:3954} INFO - Setting next_dagrun for my_dag to 2024-05-12 00:00:00+00:00, run_after=2024-05-13 00:00:00+00:00
[2024-05-12T12:39:07.507+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/script.py took 0.527 seconds
[2024-05-12T12:39:38.146+0000] {processor.py:161} INFO - Started process (PID=160) to work on /opt/airflow/dags/script.py
[2024-05-12T12:39:38.146+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/script.py for tasks to queue
[2024-05-12T12:39:38.148+0000] {logging_mixin.py:188} INFO - [2024-05-12T12:39:38.147+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/script.py
[2024-05-12T12:39:38.593+0000] {processor.py:840} INFO - DAG(s) 'my_dag' retrieved from /opt/airflow/dags/script.py
[2024-05-12T12:39:38.611+0000] {logging_mixin.py:188} INFO - [2024-05-12T12:39:38.611+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-12T12:39:38.629+0000] {logging_mixin.py:188} INFO - [2024-05-12T12:39:38.629+0000] {dag.py:3954} INFO - Setting next_dagrun for my_dag to 2024-05-12 00:00:00+00:00, run_after=2024-05-13 00:00:00+00:00
[2024-05-12T12:39:38.675+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/script.py took 0.532 seconds
[2024-05-12T12:40:09.358+0000] {processor.py:161} INFO - Started process (PID=162) to work on /opt/airflow/dags/script.py
[2024-05-12T12:40:09.359+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/script.py for tasks to queue
[2024-05-12T12:40:09.360+0000] {logging_mixin.py:188} INFO - [2024-05-12T12:40:09.360+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/script.py
[2024-05-12T12:40:09.809+0000] {processor.py:840} INFO - DAG(s) 'my_dag' retrieved from /opt/airflow/dags/script.py
[2024-05-12T12:40:09.828+0000] {logging_mixin.py:188} INFO - [2024-05-12T12:40:09.828+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-12T12:40:09.846+0000] {logging_mixin.py:188} INFO - [2024-05-12T12:40:09.845+0000] {dag.py:3954} INFO - Setting next_dagrun for my_dag to 2024-05-12 00:00:00+00:00, run_after=2024-05-13 00:00:00+00:00
[2024-05-12T12:40:09.893+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/script.py took 0.539 seconds
